---
#author: Fran\c{c}ois Michonneau
title: A data-driven approach to identifying knowledge gaps in marine invertebrates biodiversity
output: pdf_document
---


# Summary

Species identification from DNA sequences is poised to revolutionize our
understanding of biodiversity, to facilitate environmental monitoring, and to
disentangle trophic networks. However, this approach relies on comprehensive
databases of accurately identified sequences.  Since its introduction in the
early 2000's, DNA barcoding, the sequencing of a common locus across taxa
bearing species-specific information, has revitalized taxonomy by facilitating
species delineation. More recently, advances in high-throughput sequencing
technologies have enabled the parallel sequencing of thousands of species that
occur in environmental samples. The approach initially developed to study
microbial communities such as the ones from guts or soil, has been modified and
extended to be applied to metazoan communities. This approach opens new research
venues as it becomes possible to estimate the diversity from species-rich
samples that were virtually inaccessible before. This is particularly true in
marine communities that include multiple phyla and organisms varying in size by
order of magnitude, making identification challenging. However, the utility of
the data generated by these projects is enhanced by the possibility of matching
the DNA sequences obtained to previously sequenced and correctly identified
species. Here, we combined information available in public biodiversity
databases to assess the proportion of identified species that have been
sequenced in the continental United States. We show that the effort is
consistent across the better-studied taxa, with `r average_prop_species_seq`% of
the fauna sequenced. Using a DNA barcode dataset of `r n_larvae` planktonic
larvae, representing `r n_esus` species, collected in coastal waters of Florida,
we find a similar proportion of species that can be assigned to a previously
sequenced species ( `r format_output(100 * prop_plankton_with_match)`%).  Our
results show that most marine species remain to be sequenced. Achieving this
goal will required a concerted effort, and resources.


<!---
the content of insect traps, marine sediments, plankton
samples, or communities settling on artificial substratum. When applied to
metazoan communities, these sequencing approaches typically target the barcoding
gene and theefore has been dubbed "meta-barcoding". These applications are also
making the public databases for these DNA barcodes all the more valuable. To
deliver on the meta-barcoding promises, it is crucial to have comprehensive
taxonomic coverage represented in the DNA sequences public databases. Here, we
investigate the proportion of species for each marine phylum that have been
sequenced at various spatial scale.
--->

# Main text



With increased anthropogenic pressure on ecosystems, it is crucial to become
efficient at documenting biodiversity before species go extinct. This is
especially important in marine ecosystems where an undefined, but large,
fraction of the biodiversity remains to be discovered especially among
invertebrates. One approach would be to build on existing knowledge to identify
gaps and target understudied areas or taxonomic groups. This is currently
difficult because the knowledge is fragmented and not always
accessible. However, recent several large-scale initiatives have been developed
to aggregate and synthesize various facets of biodiversity knowledge. For
instance, the Biodiversity Heritage Library aims at making the biodiversity
litterature openly available, the World Registry of Marine Species provides a
database of the validity and relationships for most marine species taxonomic
names, and iDigBio digitizes and makes available on the web museum
specimen-associated data. Our knowledge about species and their geographical
range comes from:

1. **Faunistic lists** synthesize and evaluate expert knowledge about a specific
   location. While limited in geographical scope, they tend to be accurate and
   comprehensive.
1. **Museum specimens** are physical proofs of the presence of a species at a
   given point in time and space. The identification can be re-assessed, but
   their contribution to biodiversity knowledge is greatly enhanced if
   associated data and metadata are digitized.
1. **DNA barcodes** useful to delineate and identify species. It requires the
   presence of a museum voucher. Only as good as the identification. Has been
   very useful to identify species complexes, especially among marine
   invertebrates.

To assess the level of digitization for biodiversity, we compared two very
comprehensive faunistic lists for the marine fauna of the US, one from the Puget
Sound (West coast) by Kozloff and the Gulf of Mexico checklist (East
coast). These areas are well studied. Overall, we found that xx% of the species
included in these lists were not found in iDigBio, and xx% in GBIF. Microscopic,
poorly studied metaozans were less represented than larger groups, but for the
latter, proportions on the East and West coast were very similar. Most species
are likely to be undersampled as xx% of the species are represented by less than
5 lots.





However, taxonomic expertise is dwindling, and there are now only
a handful of natural history museums with active marine invertebrates
collections. Museum collections have been recording our knowledge about species
and their geographical ranges, but until recently, this information was
fragmented across institutions and difficult to synthesize. With collection
digitization and other large-scale projects have aimed at gathering biodiversity
information, public databases can now be used to enable synthesis and highlight
knowledge gaps. Here we use these data to synthetic the state of knowledge for
marine invertebrates, and demonstrate that the stage is set to combine these
data sources to inform where future efforts for documenting biodiversity.

We combined `r nrow(global_idigbio_species_list)` museum records for marine
invertebrates collected in the continental US representing
`r length(store_idigbio_records()$list())` phyla and
`r length(unique(global_idigbio_species_with_worms_info$worms_valid_name))`
species. Most samples were collected on the continental
shelf (Fig. 1). For the East coast, species richness is the highest in South Florida but
it is relatively uniform along the coastline. For the West coast, the Salish
Sea, the Bay Area, and Southern California have the highest richness (Fig. 2).

The number of sampled species is vastly underestimated.
`r prop_singleton_species`% of the species are known from a single
record. Futhermore, the strong correlation between the number of samples
collected and the species richness suggests that any new sampling effort
will lead to new records (Fig. 3).

DNA barcoding is one approach that can be used to alleviate the discrepancy
between the increased need for biodiversity inventories while taxonomic
expertise is dwindling. DNA barcoding aims at assembling a comprehensive
database of DNA sequences that are associated with accurately identified
vouchered specimens. Future identifications can then rely on this database. DNA
barcoding can also facilitate species delineation, in turn, increasing the
accuracy of the identifications.

Recent developments in high-throughput sequencing technologies are giving new
applications to these DNA barcode databases. Meta-barcoding enables the parallel
sequencing of thousands of species that occur in environmental samples.  This
approach opens new research venues as it becomes possible to estimate the
composition of species-rich samples that were virtually inaccessible before
(e.g., fish gut content, marine sediment samples, plankton tows). The ability to
use DNA to identify members of these communities relies on comprehensive DNA
barcode databases. To assess the proportion of recorded species in the US that
has been sequenced, we searched for each species occurring in museum collections
for possible matches in BOLD (which also includes GenBank records). We found
that the proportion of sequenced species is consistent across the better-studied
taxa (overall `r average_prop_species_seq`%, Fig. 4). This proportion is consistent
across phyla despite important differences in species richness, and taxonomic
tradition. We also found that phyla representative of the meiofauna are
under-represented, and very few specimens and sequences are available.

Identified specimens from museum collections could be used to complete these
datasets. Traditional approaches relying on Sanger sequencing might be limiting as
many species have not been collected recently enough to guarantee good quality DNA
(`r prop_species_not_collected_since_1990`% have not been collected since
1990). However, high-throughput sequencing could offer alternatives to recover
sequences from older specimens.

Besides data-mining museum records to identify knowledge gaps, new sampling
approaches can be used in the field to rapidly assess how much of the fauna is
known. Meta-barcoding and environmental DNA approaches appear promising to
sample _en masse_ species that are found in a given location. Because most
marine invertebrates have a planktonic larval stage, they may co-occur in the
water column, allowing to sample species across habitats. To illustrate this
approach, we individually DNA barcoded `r n_larvae` planktonic larvae for DNA
barcoding, collected over the course of year, representing `r n_esus` species,
collected in coastal waters of Florida. We found that
(`r format_output(100 * prop_plankton_with_match)`%, Fig. 4) can be assigned
to a previously sequenced species. Similar proportions have been obtained from
individually barcoded species that occur on ARMS. However, this proportion
dropped to less than 5% for smaller organisms sequenced with meta-barcoding.

The species identifications in BOLD are far from perfect. Of the
`r sum(prop_plankton_with_species_match_by_phylum$n_esu_match)` species that have a
match in public database,
`r format_output(prop_plankton_with_multi_species_match * 100)`% have been
identified as more than one taxon in BOLD. A few of these records might represent
species complexes, but most illustrate the difficulty to identify precisely
marine invertebrates.

<!-- Studying biodiversity and identifying knowledge gaps about species and their -->
<!-- distributions need to become more data-driven to target under-studided areas and -->
<!-- taxonomic groups. To survive the biodiversity crisis, we need -->
<!-- inter-disciplinarity among -->
<!-- scientists  only need to be taxonomists that can reliably -->
<!-- identify species, but also molecular biologists. -->


![Number of museum records for marine inverterbrates](figures/map_sampling_effort.pdf)

![Number of species of marine invertebrates](figures/map_diversity.pdf)

![Relation between the number of specimen and the number of species collected](figures/plot_sampling_effort.png)

![Proportion of species occuring in natural history collections that have been DNA barcoded](figures/proportion_barcoded.pdf)


<!--

The number of species that occurs in the sea remains uncertain.


While plankton is recognized for its biological importance (dispersal, food
source for many species), and can be used as an ecological indicator for
pollution or global change, identification of planktonic larvae at the species
level from morphological information is generally not possible because of the
paucity of informative characters to base the identifications on, and the lack
of knowledge of the appearance of the larval stages for most species. DNA
barcoding can bridge this knowledge gap by using genetic information to identify
larvae at the species level. These identifications are only possible if the
species have already been sequenced, and the sequences deposited in public
repositories. The accuracy of the identification relies on the expertise of the
taxonomists who identified the specimens that have been sequenced.

We collected over the course of a year planktonic larvae of marine invertebrates
in a coastal marsh of the Atlantic coast of Florida. One of our goals was to
identify these larvae at the species level using DNA barcoding. We photographed
and sequenced xxx larvae, representing an estimated xxx species. These data are
available from BOLD and can be browsed interactively at
http://plankton-atlas.net. Using records from public databases (BOLD, GenBank),
we could confidently assign only xxx% to species that had already been
sequenced. On average, these taxa were sequenced xxx km away from our study
site.

Florida is one of the most populated states, with a long tradition of studying
its marine life.  xxx universities have marine research programs, and xxx marine
laboratories. How does this proportion of xxx% identified species compare with
other places in Florida, the East coast of the US, the continental US, and the
world. In other words, how much of the species of marine invertebrates we know
about have publicly available DNA barcodes?

To estimate this number, we assembled species lists of marine invertebrates for
the continental US by querying iDigBio for each marine phylum. We only kept
records for specimens identified with a valid binomial name (specimens
identified at the genus level or higher were excluded from the analysis). We
used WoRMS to ensure that the species names were valid, and to disambiguate
possible synonyms. For each species, we retrieved avaiable DNA barcoding
sequence data from BOLD, which also includes DNA barcodes deposited in
GenBank. Of the xxx marine invertebrate species found in iDigBio, xxx% had DNA
barcodes available in public databases. These proportions varied by phylum, but
for the conspicuous and better studied groups (Porifera, Cnidaria, Mollusca,
Echinodermata), they were surprisingly consistent, and is a close estimate to
the proportion of species that have been sequenced for the United
States. Furthermore, we also looked for the proportion of species known to occur
in the Indian River Lagoon for which DNA barcodes are available, and again the
proportion of sequenced species (xxx%) is comparable.

Except for the few phyla that are known to include only a small number of
species (Cycliophora, Phoronida, Chaetognatha), microscopic groups, typically
component of the meiofauna, are disproportionally under-represented in museum
collections, and none or few species are available in public databases.

Our results highlight that most of our knowledge of marine biodiversity come
from large expeditions carried in the late 19th century and after World War
II. Since then, sampling effort has dramatically decreased, and taxonomic
expertise is waning. Yet, our ability to assign DNA sequences to species names
will greatly improve the value of the data generated by meta-barcoding studies.

and reflect that a large fraction of the diversity of
marine invertebrates remain to be sequenced.






















Marine invertebrates are the biodiversity. All but one metazoan phyla occur in
the sea. Of the xx described species, yy are marine invertebrates. They
represent a multiple array of shapes, forms, and adaptations to all marine
environments. Yet, they are proportionally understudied. Here, we analyze
publicly available data from biodiversity databases to demonstrate that the
abandonment of traditional taxonomic practices, the lack of support for natural
history collections, is hampering our ability to use marine invertebrates as
indicators of ecosystem health, the effect of global change, the identification
of model organisms, and ... Technological advances are however making easier to
identify knowledge gaps, the potential to identify where new species occur, and
document diversity.


# The role of DNA barcoding

The use of molecular data for species identification and delineation is arguably
one of the most important advances in taxonomy. Especially with DNA barcoding
that proposes to use a common genetic marker across large taxonomic groups (the
mt marker COI for animals, ITS for fungi, and xx for plants). Because these
markers have typically low intra-specific variation, and differ noticeably
between even closely related species, they can be used to identify species and
unravel species complexes.

Species identification relies on reference databases, where sequences are
associated with accurately identified individuals, held in natural history
collections. When taxonomic sampling is comprehensive, and enough individuals of
each species are sampled to be representative of the standing genetic variation
in the population, DNA barcoding can be a very accurate tool with xx-yy% error
rate.

The utility of these reference databases is getting revived with the development
of meta-barcoding techniques, where thousands of species can be sequenced in
parallel on high-throughput sequencing platforms. If methods exist to estimate
the number of species these samples contain based on the level of genetic
divergence observed among the sequences, being able to match these sequences to
identified species based on their match to a sequence in a public database would
increase the potential applications for these approaches.

Marine invertebrates are however disproportionally under-represented in DNA
sequence databses. And they have a higher rate of misidentification. There are
however some pitfalls to this number. It might actually be even higher given
that they have been identified by a single taxonomist given the lack of
redundancy in expertise.

To illustrate the consequences of having so few barcodes for invertebrates, if
we take the faunistic list of IRL, only about 20% of the species known to be
occuring in the area actually have DNA sequences available for them. On average,
each species has been sequenced xx times, and from locations on average yy km
away from IRL. Given the high propencity for cryptic species among marine
invertebrates, it is possible that ...

We also sequenced xx larvae occurring in the plankton at the Whitney Marine
Lab, representing yy species (estimated from genetic distances). Of these yy
species, only zz% had good matches in BOLD/GenBank. In both cases, taxa1, taxa2,
and taxa3 are the most underrepresented in BOLD.


# The waining of taxonomy

Total number of collections in the US, how many have marine invertebrate
collections, how many have active curators.




We don't know how many species are in the sea. Estimates vary widely. Difficult
to sample because diversity of habitats, each requiring special sampling
techniques. Meanwhile, taxonomic expertise is decreasing despite efforts to keep
it alive (PEET).

To get around issue of sampling different habitat, sample coastal
meroplankton. Most species go through planktonic phase, therefore common habitat
to sample them all. The issue is that they look very different from adults, and
only DNA can be used to identify them.

Barcoding is not perfect, but can be very powerful to identify the larval forms
(e.g. one of the first paper was sampling caterpillars). Potentially differences
in larva that are not seen in adults and could be additional characters to tell
species appart.

Advantage of sampling plankton is that a single habitat is sampled. Not enough
known about heterogeneity in spatial distribution.

We sampled plankton bi-weekly for a year, and sequenced the larvae
found. Sequenced the barcoding gene and tried to match how many species we could
identify based on records in public databases. Overall, low percentage of
match. And matches also led to several species that have been collected but that
have not been identified to an appropriate level.

To compare the results we obtained here, and see how they could compare to other
habitats, we downloaded the species list from Indian River lagoon inventory. It
is the most comprehensive checklist in Florida, and has been developed over many
years. We asked how many of the species occurring in the Indian River Lagoon have
already been sequenced. We did the matching based on the species names. If we
find higher proportion of sequenced species, it means that plankton gives access
to species that are difficult to sample as adults. If lower proportion, it means
that we don't have many of the adults sequenced at Whitney.

Using plankton to sample biodiversity could lead to a speedup of
documentation. It would allow the identification of taxa/habitats that need
special attention. If representative of adult communities, could also be a
useful tool to monitor status of ecosystem through time.

Better database, can also be used to estimate the proportion of really dark
planktonic taxa.

Understand better population connectivity, currents



Importance of plankton in connecting populations. Important as food source.

Swimming in the dark in the age of big data

Biodiversity is in the sea. All phyla but one. New lineages of eukaryotes,
viruses...

We are at a time where it becomes cheaper and easier to delineate species. To
sort through specimens to identify species. So taxonomists can focus on
studying/describing new species.

Granularity of data not sufficient to be able to detect changes in the
environment, shifts in species ranges, changes in trophic structure. Understand
effect of small animals.

Rely on high school students, citizen science initiatives to help with labor
intensive part.

Oceans are fascinating.

Bioprospection

focus on a few taxa but lots of groups we know nothing about.

Strong focus on vertebrates. We are building theories and making conservation
approaches on a small fraction of the tree of life.

Trait data

Species Distribution

--->
