---
#author: Fran\c{c}ois Michonneau
title: "The state of knowledge for marine invertebrate biodiversity in the continental US"
date: ' ' #'`r format(Sys.time())`'
output:
  bookdown::pdf_document2:
    #includes:
    #  before_body: ../template/doc_prefix.tex
    #  in_header: ../template/preamble.tex
    keep_tex: yes
    latex_engine: pdflatex
    number_sections: no
    toc: no
fontsize: 12pt
link-citations: yes
csl: nature.csl
bibliography: /home/francois/Library/dark-plankton.bib
---

```{r setup, echo=FALSE, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```

<!-- use Pimm+2014 intro as model on using short informative sentences to build -->
<!-- 1st paragraph. -->


<!-- * how do we conserve animal species that we can't see, we don't find pretty, don't have eyes -->
<!-- * Most species are rare, especially in very diverse places,  and the degradation of habitats makes it even more difficult to find these rare species. -->

## Abstract

Museum collections have been recording our knowledge about species and their geographic ranges, but until recently, this information was fragmented across institutions and difficult to synthesize. The digitization of natural history collections and other biodiversity information systems can now be used to enable synthesis and highlight knowledge gaps. Here we use these data to synthesize the state of our knowledge for marine invertebrates in the continental United States. We show that current knowledge and sampling efforts are inadequate given the diversity of the fauna and the geographic extent of US. While recent high-throughput sequencing technologies promise to greatly speed up the documentation of marine species and their ranges, its value will be greatly improved by anchoring the results into existing DNA sequence data and associated vouchers. However, the best documented phyla have at most 25% of their species sequenced for the most widely used DNA barcoding gene, cytochrome oxidase I (COI). More sampling, coordinated at the scale of the US, is needed to improve our knowledge of species and their geographic ranges. This data is needed to detect the impact of climate changes and other anthropogenic strains to the environment. Advances in computing, information systems, and sequencing technologies, have set the stage to inform where future efforts for documenting biodiversity should be applied.


## Introduction

With increasing impact of human activities on the biosphere, we need efficient and accurate approaches to document changes in species ranges and community composition. While terrestrial ecosystems have received considerable attention, the baseline of healthy and resilient marine ecosystems are not well understood [@Worm2006], and modeling the impact of human activities has been limited to a few taxonomic groups [@Tittensor2010]. For marine organisms, because of the diversity of their complex life history strategies, their size range, and the difficulties associated with sampling their habitats, we have limited information about which species live where, or what portion of their diversity remains to be discovered and described [@Mora2011]. Additionally, coordinated marine ecosystem monitoring initiatives have been lacking [@Duffy2013,@Borja2016] (but see MarineGEO [@Duffy2016marinegeo] and PISCO), making it difficult to monitor changes in species distributions and abundances. While most documented global extinctions of marine species are vertebrates, many others species may have been lost unnoticed because they were not known [@Sala2006]. There is therefore a pressing need to improve our knowledge of marine systems before their functioning are irreversibly altered, and species go extinct.

Research attempting to understand how much we know about marine biodiversity has focused on estimating the total number of species that inhabits the oceans, and which fraction of this diversity has been named [@Appeltans2012,@Costello2013]. Undoubtedly, the development of molecular methods to investigate species limits has reinvigorated taxonomy, and contributed to elucidating cryptic species complexes [@Knowlton2000]. Thus, it has been argued that molecular data can speed up the enterprise of discovering and naming species [@Blaxter2016]. As species remain the most convenient unit to measure biological diversity, understanding how much has been done, how much remains to be done, and how we can speed up the description of species diversity is crucial. However, to understand the role of these species in their environment, we need to document the ecology of these species [@Cernansky2017]. For instance, to be able to predict how their geographic ranges will be affected by global change, we need detailed information about their environmental requirements. Occurrence data for marine species could be used for environmental niche modeling of their geographic and bathymetric distributions but the proportion of species for which we have enough data to do so is unknown.

Until very recently, our knowledge of biodiversity was scattered among heterogeneous sources (e.g. natural history collections, publications, expert knowledge), difficult to access, and therefore difficult to synthesize. The development of public databases for specimen records (e.g. iDigBio [@Page2015]), species occurrences (e.g., OBIS [@Zhang2002]), genetic data (e.g., GenBank [@Benson2013], BOLD [@Ratnasingham2007a]), and registers of species names (e.g., WoRMS [@Costello2013b]), now allow us to evaluate what and how much we know. Here, we aggregate publicly available data for marine invertebrates that occur within the limits of the exclusive economic zone (EEZ) of the continental United States (US), as well as the Gulf of Mexico (GOM) and the Pacific Northwest (PNW), to highlight gaps in our knowledge that provide a road map for important new directions in global research. We show that while public databases are capturing a significant portion of our knowledge about species richness, most species are known from only a few specimens and a few locations. We emphasize that our understanding of marine biodiversity, and the impact of human activities on communities require not only to get a more accurate list of the species that inhabits the oceans, but more comprehensive data on species distributions and abundances are needed. These goals can only be achieved with additional cost-effective sampling and a commitment to training more taxonomists and experts that can identify and interpret the material collected.

Here, we focus on the continental US because the marine biodiversity is extensively documented as the result of a long history of studies by governmental, academic, and private institutions [@Fautin2010]. The continental US includes a wide diversity of marine ecosystems from the Florida Keys, to the Gulf of Maine on the East Coast; and from the arid Southern California to the oceanic shores of the Pacific Northwest on the West coast. With estuaries, bays, sandy beaches, rocky shores, and coral reefs, the US coastlines harbors many biota. While the continental shelf extends for several hundred kilometers on the East coast, it is narrow on the West coast. This diversity of habitats and bathymetric profiles, combined with efforts at the national level to make data publicly available, make this region especially suitable to evaluate how much we know about marine biodiversity using biodiversity repositories. Additionally, apart from the syntheses that emerged from the Census of Marine Life efforts [@Costello2010,@Fautin2010], no study has yet attempted to synthesize the current state of knowledge of marine biodiversity across the continental US, and this study is the first using data from public databases.


## Results

### Comparison with Regional Lists by Geography

We extracted `r format(sum(gom_worms$is_marine, na.rm=TRUE), big.mark=",")` and `r format(sum(kozloff_species$is_marine, na.rm=TRUE), big.mark=",")` marine invertebrate species from species checklists from the Gulf of Mexico [@Tunnell2009] and from the Pacific Northwest [@Kozloff1996] respectively. For the Gulf of Mexico, biodiversity databases included more species for the same geographic area as the one used by the authors of the checklist (iDigBio and OBIS combined: `r format(dplyr::n_distinct(combined_gom_records$worms_valid_name), big.mark=",")` species; iDigBio only: `r format(n_distinct(idigbio_gom_records$worms_valid_name), big.mark=",")` species, $n =$  `r format(nrow(idigbio_gom_records), big.mark=",")` records; OBIS only: `r format(n_distinct(obis_gom_records$worms_valid_name), big.mark=",")` species, $n=$ `r format(nrow(obis_gom_records), big.mark=",")` records). For the PNW, fewer species than included in the regional checklist were recovered from biodiversity databases (iDigBio and OBIS combined: `r format(dplyr::n_distinct(combined_pnw_records$worms_valid_name), big.mark=",")` species; iDigBio only: `r format(dplyr::n_distinct(idigbio_pnw_records$worms_valid_name), big.mark=",")`  species, $n=$ `r format(nrow(idigbio_pnw_records), big.mark=",")` records; OBIS only `r format(dplyr::n_distinct(obis_pnw_records$worms_valid_name), big.mark=",")` species, $n=$ `r format(nrow(obis_pnw_records), big.mark=",")` records). The combination of the species listed in these three sources leads to `r n_distinct(total_gom_records$worms_valid_name)` species for the GOM and `r n_distinct(total_pnw_records$worms_valid_name)` species for the PNW.

```{r compare_database_coverage, fig.cap='Overlap between the regional species list (list), records in iDigBio (idigbio), records in OBIS (obis), and records in BOLD (bold) for the Gulf of Mexico (left) and the Pacific Northwest (right). This plot is generated using the UpSet technique[@Lex2014;@Conway2017], which uses a matrix-based layout to represent intersections of sets and their sizes. The rows represent the sets, the columns represent the intersections, and each set that is part of the intersection is marked with a black-filled circle in the cells underneath. Black lines connect the sets that are part of the intersections. (\\#fig:database-coverage)', echo=FALSE}
knitr::include_graphics("figures/database_coverage.pdf")
```

Digitization of species richness is higher for the GOM compared with the PNW. A higher proportion of the species included in the regional list for the GOM are captured in biodiversity databases than for the PNW. While `r format_output((3890 + 2663 + 1025 + 316 + 102 + 49 )/n_distinct(total_gom_records$worms_valid_name) * 100)`%  of the species included in the checklist for the GOM have records in at least one of the biodiversity database (Fig. \@ref(fig:database-coverage)A), `r format_output((352 + 292 + 281 + 235 + 76 + 38)/n_distinct(total_pnw_records$worms_valid_name)*100)`% do for the PNW  (Fig. \@ref(fig:database-coverage)B). Additionally, a higher proportion of species occur in the biodiversity databases but are not included in the regional checklist for the GOM compared with the PNW (GOM: `r format_output((1253 + 848 + 556 + 216 + 190 + 124) / n_distinct(total_gom_records$worms_valid_name)*100)`%, $n =$ `r n_distinct(total_gom_records$worms_valid_name)`; PNW:
`r format_output( (281 + 151 + 145 + 80 + 78 + 45) / n_distinct(total_pnw_records$worms_valid_name)*100)`%, $n =$ `r n_distinct(total_pnw_records$worms_valid_name)`).


```{r compare_database_overlap, fig.cap='Proportions of the total number of species by phylum that are absent from faunistic lists but are in iDigBio or OBIS ("Not in list"), and absent from iDigBio or OBIS but are in the faunistic lists ("Not in databases"). The numbers represent the number of species. (\\#fig:database-overlap)'}
knitr::include_graphics("figures/database_overlap_percent.pdf")
```


<!-- need to separate across all phyla statements and by phylum statements -->
We investigated the number of species for each phylum that contributed to the discrepancies between the regional species lists and the database records. We found that the more diverse phyla contributed the most to the differences across the data sources. OBIS tends to be closer to the full record set, but iDigBio has more species for the Gulf of Mexico that are not included in the list (Fig. \@ref(fig:database-overlap)). Many species of mollusks, arthropods, and annelids, known to occur in the Gulf of Mexico or in the Pacific Northwest, have no digital records in these geographic areas.


<!-- Regional faunistic lists assembled and curated by taxonomic experts provide a comprehensive and detailed account of species that are known to occur in a given geographic area. We assessed how faunistic lists assembled over decades by specialists compare with records in public databases for the same geographic area. -->


Several factors may contribute for species to be included in the regional lists but to not have records in biodiversity databases. The first category of biases are linked to property of the species affecting the likelihood of having specimens preserved in museum collections. For instance, rare species may never have been collected or have only been collected a few times, and the corresponding records have not been digitized yet. Species that are difficult to preserve because they are large, very small, or fragile, are deposited in museums less frequently. The other category of biases is associated with data quality for these records. Some of these species may be in the databases but do not have geographic coordinates associated with them, and therefore would not be included in the results of our query.

Missing geographic information contribute to the discrepencies observed.  Using iDigBio we found that for the `r format(not_really_in_database$idig_gom_n_spp_not_in_db, big.mark=",")` species that do not appear within the geographic range of the Gulf of Mexico but are on the list, `r format_output(not_really_in_database$idig_gom_p_spp_not_in_db*100)`% are completely absent from iDigBio, while for the `r format(not_really_in_database$idig_gom_n_spp_no_coords, big.mark=",")` species that do appear in iDigBio, `r format_output(not_really_in_database$idig_gom_p_spp_no_coords*100)`%  do not have geographic coordinates associated with them, and for the ones that do have geographic coordinates, `r format_output(not_really_in_database$idig_gom_p_spp_within_usa*100)`% ($n=$  `r format(not_really_in_database$idig_gom_n_spp_within_usa, big.mark=",")`) fall within the US waters. These numbers indicate that missing geographic coordinates reduces the overlap between the list of species from the regional list and the databases. Additional digitization of collections, including the capture of their geographic coordinates information, are needed to improve biodiversity knowledge at regional scales. For instance, one can expect that the digitization coverage for the Gulf of Mexico will greatly increase once records from the Florida Wildlife Research Institute collections (St Petersburg, FL) become integrated into biodiversity information systems.

Two factors are most likely to contribute to explain why some species are in databases but were not considered by the authors of the checklists: (1) the species have been documented for the first time since the publication of the checklist, (2) the species records were not accessible or known to the authors.

<!-- include also proportion of species not in list -->
Recently recorded species contribute modestly to this discrepency, as `r sum(recently_collected$gom_in_dbs$min_year > 2001)` species (`r format_output( sum(recently_collected$gom_in_dbs$min_year > 2001)/n_distinct(total_gom_records$worms_valid_name) * 100)`% of the total species richness) have been collected for the first time since the publication of the list in the Gulf of Mexico, and `r sum(recently_collected$pnw_in_dbs$min_year > 1996)` species (`r format_output(sum(recently_collected$pnw_in_dbs$min_year > 1996)/n_distinct(total_pnw_records$worms_valid_name) * 100)`% of the total species richness) for the Pacific Northwest.

The lack to tools for the authors of these checklist to have access to museum records across institutions contributed most to explain why some species were not included in the checklists. These records were difficult to discover before digitization which, for the most part, has occured after the compilation of these checklists in most cases.

### What do we know about marine invertebrates in the US?

<!-- add sentence -->
<!-- separate idigbio and (and add) obis in the results -->

We retrieved `r format(n_distinct(combined_species$worms_valid_name), big.mark = ",")` species (`r n_distinct(na.omit(combined_species$worms_phylum))` phyla) of marine invertebrates in the continental US EEZ by combining  `r format(nrow(idigbio_records), big.mark=",")` museum records from iDigBio (`r format(n_distinct(idigbio_records$worms_valid_name), big.mark = ",")` species, `r n_distinct(na.omit(idigbio_records$worms_phylum))` phyla), and `r format(nrow(obis_records), big.mark=",")` records from OBIS (`r format(n_distinct(obis_records$worms_valid_name), big.mark=",")` species,
`r n_distinct(na.omit(obis_records$worms_phylum))` phyla). Extant marine invertebrates occurring in the US represent less than 0.4% of the total iDigBio records, as it currently holds records for over 106 million specimens, and they represent about 1% of the records that occur in the United States (about 33 million records in iDigBio). Marine invertebrates that occur in the US represent approximately 4% of the 48 million occurrence records in OBIS.

<!-- need to add latitudinal gradient map -->
Most samples have been collected on the continental shelf (Fig. \@ref(fig:map-diversity-sampling)A). On the East coast, species richness is highest at low latitudes, while it increases towards high latitude on the West coast (Fig. \@ref(fig:map-diversity-sampling)B).


```{r map-diversity-sampling, fig.cap='Spatial distribution of the number of samples (A) and species richness (B) for marine invertebrates based on records retrieved from iDigBio and OBIS within the limits of the Exclusive Economic Zone for the continental USA. (\\#fig:map-diversity-sampling)'}
knitr::include_graphics("figures/combined_map_sampling.pdf")
```

Spatial patterns of species richness are strongly confounded by sampling effort. Samples of <100 records per cell commonly have as many species as records (Fig. \@ref(fig:correlation-sampling)).

```{r correlation-sampling, fig.cap='Relation between number of records and number of species for each cell of the map used in Fig. \\@ref(fig:map-diversity-sampling). The black line represents the 1:1 relation. (\\#fig:correlation-sampling)'}
knitr::include_graphics("figures/plot_sampling_effort.pdf")
```

Underscoring the inadequacy of the sampling effort, `r prop_singleton_species`% of the species are known from a single record. Another indication of this inadequacy is the relation between the number of collected specimen and the number of species recorded through time (Fig. \@ref(fig:idigbio-through-time)). Initial explorations in the late ninetieth century led to an initial exponential growth in the knowledge of the fauna, that tapered off, until the 1970s when intensive collecting efforts led to an increase in the recording of new taxa (Fig. \@ref(fig:idigbio-through-time)). Since then, the slowing down of collections has been accompanied with a slowing down in the discovery of new records: `r n_spp_comparison$n_spp_2017_2000` species have been documented for the first time since 2000 (`r format_output(n_spp_comparison$n_spp_2017_2000/18)` species.year^-1^), while `r n_spp_comparison$n_spp_1970_1960` have been documented between 1960 and 1970 (`r format_output(n_spp_comparison$n_spp_1970_1960/11)` species.year^-1^). The strong correlations between (1) the number of samples collected and the species richness (Fig. \@ref(fig:correlation-sampling)), and (2) the number of specimens collected per year and the number of species recorded for the first time (Fig. \@ref(fig:idigbio-through-time)C), suggest that any new sampling effort will lead to new observations and new species. <!-- separate issues of sampling is needed, and large-scale sampling is best way of doing it --> Coordinated large-scale biodiversity expeditions continue to be needed to improve our understanding of the diversity of species and their spatial distributions. The US marine EEZ is relatively well-known compared to many areas of the world ocean, where studies show that undescribed biodiversity dominates. For example, a recent survey in Western Australia reported that 94.6% of crustaceans and 72% of polychaetes taken at 135 sites were undescribed [@Poore2014].

```{r idigbio-through-time, fig.cap='Cumulative number of samples (A) and species (B) of marine invertebrates recorded through time based on iDigBio records. Relation between number of samples collected, and number of species recorded for the first time (C), each point represents numbers for a given year, dashed line show the 1:1 and dotted line the 1:100 ratios for number of species recorded for the first time to the number of samples. (\\#fig:idigbio-through-time)'}
knitr::include_graphics("figures/idigbio_through_time.pdf")
```

Environmental conditions are among the primary determinants of species distributions and can thus be used to estimate the response of organisms to climate change. Predictive ecological models forecasting changes in distributions are training on environmental conditions encountered by species in their current distributions. To obtain reliable results, models require at least 10, but ideally 30 or more occurrence records per species [@Wisz2008]. Yet at present, `r prop_10_specimens_species`% of species are represented by $\leq$ 10 records, and `r prop_30_specimens_species`%  by $\leq$ 30 records based on the combined records in iDigBio and OBIS.


Further biases exist across phyla and body size. Large-bodied and easily acccessible (e.g. dried specimens like shells, corals, echinoderms) are overrepresented in museum collections, compared with small less accessible organisms, such as the meiofauna. Of `r n_idigbio_phyla` phyla represented in iDigBio, `r records_rare_phyla$n_rare_phyla` have <500 records, and make up `r records_rare_phyla$prop_recs_rare_phyla`% of the iDigBio records. This underrepresentation is manifested at the species level, while `r records_rare_phyla$prop_spp_rare_phyla_collected`% (n = `r format(records_rare_phyla$total_spp_rare_phyla_collected, big.mark=",")` total species) of the global number of species for the `r records_rare_phyla$n_rare_phyla` phyla occur in iDigBio, this proportion rises to `r records_rare_phyla$prop_spp_common_phyla_collected`% (n = `r format(records_rare_phyla$total_spp_common_phyla_collected, big.mark=",")`). Yet, these undersampled taxa, especially those of small body size, are often the most abundant, and are likely to play an important role in the detection of changes in environmental quality[@Bik2012]. Additionally, the ecological implications of changes in the structure and composition of the communities of these small-bodied organisms, on higher trophic levels and ecosystems are largely unexplored.

The proportion of samples that are identified to species show a marked decline in recent decades in most taxa other than mollusks, demonstrating limiting taxonomic effort and expertise (Fig. \@ref(fig:identification-through-time)). There is a marked break in the central trend arond the 1970's indicating that it now takes decades, on average, to identify samples deposited in collections. For diverse groups such as annelids or arthropods, the proportion of specimens identified to the species-level is now below 30%. The number of scientists describing new species might be increasing globally [@Costello2013], but taxonomic experts continue to be needed to identify specimens recently collected in the US.

```{r id_though_time, fig.cap='Proportion of marine invertebrate specimens collected through time identified at the species level. Trend lines are  B-splines of the 3rd degree polynomial. For all groups, we observe a decline in the proportion of specimens collected identified at the species level suggesting a lack of expertise to identify precisely what is collected. (\\#fig:identification-through-time)'}
knitr::include_graphics("figures/idigbio_identification_level_through_time.pdf")
```


### Molecular data for the future of biodiversity discovery and monitoring

Since its introduction in the early 2000s, DNA barcoding, has revitalized taxonomy by facilitating species delineation [@Meyer2005-plos]. More recently, advances in high-throughput sequencing technologies have enabled the parallel sequencing of thousands of specimens in environmental samples (eDNA) cost effectively [@Leray2015], making it possible to estimate diversity and identify taxa from species-rich samples at unprecedented scales. With decreasing sequencing costs, and the prospect of automating sample collection, these technologies may soon be used routinely to monitor biodiversity continuously [@Bohan2017]. However, the utility of the data generated by these projects is limited by the paucity of reliably identified DNA barcode sequences that can serve for matching and identification of eDNA transcripts [@Leray2015]. Thus insights from eDNA technologies depend on the level of completeness and accuracy of comparative, species-level DNA barcode coverage.

<!-- include results earlier? -->

To estimate the proportion of the marine invertebrate diversity for which DNA barcodes are available, we queried BOLD (Barcode of Life Datasystem) for all species listed in the regional lists for the Gulf of Mexico, the Pacific Northwest, as well as all the records for marine invertebrates that occur in the EEZ of the US found in iDigBio.


<!-- add overall mean number before getting into details -->
The proportion of species within the US EEZ that have DNA barcodes varies from `r format_output(stat_barcoding$min_eez_value * 100)`% for `r stat_barcoding$min_eez_phylum` to `r format_output(stat_barcoding$max_eez_value * 100)`% for `r stat_barcoding$max_eez_phylum` (Fig. \@ref(fig:proportion-barcoded)). The GOM has comparable coverage than for the EEZ, while the PNW is better barcoded with >30% species-level coverage across each of the five most diverse phyla. This is driven in part by the active Canadian barcoding effort in British Columbia. In 2012, it was estimated that, at a global scale, less than 3% of the marine species had been sequenced [@Appeltans2012], and that this number was even lower in some areas (e.g. 2.6% for the Antarctic [@Grant2009]). Thus the US fauna is generally better characterized than other parts of the world.

The above estimates are based on species names, but many traditional species include multiple cryptic species [@Knowlton2000;@Meyer2005-evolution]. For example, up to `r format_output(100 * stat_bold$p_spp_multi_bin)`% of `r format(stat_bold$n_spp_not_singleton, big.mark=",")` species represented by more than 1 record in BOLD include cryptic species, as denoted by assignment to  more than 1 BIN (BOLD Index Number, operational taxonomic units that closely resemble species [@Ratnasingham2013]). Specimens identified under the same species but sampled from distant locations, are especially prone to representing cryptic species. Geographic information is missing for most records (`r format_output(100*stat_bold$p_no_coords)`% of the `r format(stat_bold$n_total_records, big.mark=",")` BOLD records analyzed here have no geographic coordinate information associated with them). It is thus crucial to ensure that accurate geographic information is provided when DNA barcodes are published, and that enough samples for each species are sequenced, to capture more accurately their geographic range and the extent of their genetic diversity.

```{r proportion-barcoded, fig.cap='Proportion of species known to occur in the US EEZ based on iDigBio records, in the Gulf of Mexico [@Tunnell2009], in the Pacific Northwest[@Kozloff1996], that have publicly available DNA barcodes. Numbers next to the bars indicate the number of species for which DNA barcodes are available. (\\#fig:proportion-barcoded)'}
knitr::include_graphics("figures/proportion_barcoded.pdf")
```


## Where do we go from there?

Based on the results of this study, we recommend a multi-level approach for both improving the quality of existing data, as well as for the efficient collection of new data.

### Improve existing data

With data aggregation of natural history collections through initiatives like iDigBio, and the compilation of a nearly complete register of marine species with WoRMS, in combination with the development of algorithms for text analysis, it is increasingly easier to validate and spell check the scientific names of the identifications of digitized museum specimens. Linking species names to identifiers in taxonomic databases would facilitate data linking, specimen discoverability, and data usability. Additionally, because species names registers like WoRMS link the history of the names and their synonymies, it is possible to dissociate the concepts of species identification with the concept of species identity, allowing researchers to compare specimens identified at different time periods, a tedious process that would have required expert knowledge in the past.

Similarly, a significant fraction of museum records have missing or inaccurate geographic information associated with them. Geographic information might be available in other forms, but needs to be digitized. While tools have been developed to convert names of locales into GPS coordinates on land [@rios2010geolocate], similar approaches are less-developed for the sea.

For both the spelling of scientific names, and the geographic information, computing tools can be developed to detect issues and propose solutions. There are opportunities to develop computing services that use publicly available data to flag and fix these issues directly from collection databases records (e.g. see [@Otegui2016] for vertebrate records). It however requires that resources are made available to museum collections in order to address these issues directly in collection databases.

Increasing the computing literacy and teaching data wrangling skills to staff of museums in order for them to manipulate and use the data from their collection database is equally critical. As data across institutions get incorporated and pooled into large biodiversity information systems, it is crucial that data standards such as Darwin Core are embraced by the community.

Another venue to improve the quality of the data in biodiversity information systems, is to develop better incentives from journal editors and funding agencies for scientists to publish their occurrence records in databases alongside their papers as it is done for genetic data with GenBank [@Huang2011e].

<!-- Specimens with accurate identifications and specimens that have been preserved in conditions such that it is possible to get DNA out of them should be prioritized, as they yield the most data that are relevant to biodiversity knowledge. -->


### Collect better data faster and more cost effectively


 However, museum specimens are critical to validate the occurrence of species [@Rocha2014], especially for marine invertebrates for which species limits based on morphology have been known to be challenged by other lines of evidence including molecular data [@Knowlton2000].

This study highlights that we have too little data about too few species of marine invertebrates. We recommend that additional coordinated field expeditions are organized to sample more comprehensively the marine environment. The relation between sampling effort and diversity from museum collections makes clear that any additional samples will improve greatly our knowledge of marine biodiversity. Concerted and collaborative efforts are needed to share expertise and limited resources.

Samples collected during these expeditions need to be photographed to document live appearance that would otherwise be lost during the preservation process. We also need to capture metadata associated with specimens to be able to understand better the physical characteristics of the habitats and the life history for these species. Specimens need to be sub-sampled to create material suitable for genomic analysis. Parts of data acquisition associated with specimens and their biota can be automated and captured digitally directly in the field, speeding up their ingestion in database collections.

Integrating metabarcoding as part of the data collection conducted in biodiversity surveys is a promising approach to screen an habitat for its potential to lead to new discoveries. For instance, by analyzing metabarcoding samples that contain planktonic larvae from a given habitat, we could get an indication of which taxonomic group would benefit most from additional sampling effort or give an indication of the proportion of species that has not been sampled yet. Additionally, data generated by metabarcoding efforts can be re-interpreted as more DNA barcodes associated with correctly identified specimens become available. Building the library of DNA barcodes in parallel of metabarcoding efforts ensures that database matches correspond to actual specimens deposited in natural history collections, ruling out possible artifacts of the method. With such approaches, the geographic distribution of species can build on and be linked to existing records rather than being limited to matching DNA sequences across studies.

Assembling species lists is a necessary but not sufficient step in our quest to understand biodiversity. We also need to understand better the geographic distribution of biodiversity, how the abundance of species vary in space and time, and their role in the functioning of ecosystems. In order to predict the impact of climate change and other anthropogenic perturbations on species distributions, we need to collect additional occurrences for species as well as the characteristics of their habitats. Among the challenges of this enterprise, we need to be able to convey taxonomic concepts for species which, for the most part, have not formally been named yet. Developing robust identifiers that can be shared across biodiversity information systems, and anchored in results from DNA barcoding or other molecular techniques, is needed to collaboratively build our knowledge of species distributions.

Metabarcoding also offers to be a promising technology to monitor changes in communities and for understanding the functioning of ecosystems. For instance, despite our limited knowledge of the diversity of communities in the meiofauna, metabarcoding has been successfully used to detect and documents changes in species assemblages following the Deepwater Horizon oil spill [@Bik2012]. Metabarcoding has also revealed important differences in the diet of three related species of fishes that co-occur at small spatial scales, providing a much more detailed understanding of trophic relations [@Leray2015b].

## Conclusion

Given the urgency of the biodiversity crisis, we need to develop data-driven initiatives to document biodiversity more efficiently, and build on our existing knowledge to collect data that will improve our understanding of marine systems.

DNA sequences have been one of the first test case on how to link biodiversity data across repositories. These initial efforts have revealed the lack of infrastructure to use data formats specifically designed for this task, the lack of ontologies available for biodiversity data, and the fragility of the identifiers [@Stucky2014; @Walls2014; @Guralnick2014]. As new types of data will be collected, added, and linked across repositories, the community needs to develop modern and robust ways to integrate these heterogeneous biodiversity data sources [@Guralnick2015; @Deck2015].

Digitization efforts are catching up on already fragmented and incomplete records. Even if technological advances can greatly speed up the documentation of species and their distributions, taxonomists and experts are needed  to collect, sort, process, and identify samples. Taxonomic expertise within the US is dwindling, and there are now only a handful of US natural history museums with active marine invertebrates collections. This lack of resources is not sustainable in the face of the challenges and discoveries that marine ecosystems have to offer.

Crowd-sourced data and citizen science initiatives have been proposed to augment data about species abundance and distributions [@Pimm2014]. Beyond the application to near-shore habitats on easily recognizable species [@Delaney2008], large-scale efforts are currently not possible for marine ecosystems given the specialized equipment needed to sample most habitats and the expertise required to identify species. There are however opportunities for citizen science to help curate biodiversity data, while educating and engaging the public with marine conservation issues. For instance citizen science has been used successfully to classify plankton images [@Robinson2017], or transcribe hand-written museum labels [@Hill2012].

Here, we reported results from the US where there is at least two centuries of biodiversity exploration data and high-levels of digitization of museum collections. Many other parts of the world need additional attention, especially in some developing countries where environmental pressure on ecosystems is even higher than in the US.

<!-- widening gap with other groups (vertebrates, plants) for which more comprehensive data is available -->

<!-- here there are 4 things: data availability (having good APIs to access the
data); data comprehensiveness (how many species are captured in the database);
data accuracy (can we trust the data that is found in the databases); data linkage -->


## Methods

The list of species for the Gulf of Mexico was obtained  from http://e-gulf.org/ [@egulf]. This database includes the species listed in a comprehensive biotic inventory of the Gulf of Mexico [@Felder2009].

The list of species for the Pacific Northwest was downloaded from:  https://www.eopugetsound.org/species/custom-lists/307 (accessed on July 5th, 2017). This is a digital version of the species included in [@Kozloff1996].


<!-- Our goals were (1) to compare  the species lists for these geographic areas assembled by experts with the records found in databases; (2) to assess the proportion of species included in the expert lists that have records in public databases. -->

<!-- First, we compared the faunistic lists from the Gulf of Mexico and the Pacific Northwest with records in public databases. we queried the iDigBio (https://portal.idigbio.org) and OBIS (http://www.iobis.org/) databases for all records they included in the same geographic boundaries as the lists. We filtered the results to only include the same higher taxonomic level taxa as in the faunistic lists, and kept only records for which the species could be matched to a species name in WoRMS. For the Gulf of Mexico, we retrieved -->

For geographic searches, we obtained the boundaries of the Exclusive Economic Zone for the United States from marineregions.org (mrgid:8456) [@mregions]. For the Pacific Northwest, Kozloff did not specify the exact geographic extent covered by his book. We inferred a polygon that encompass part of the Oregon coast to the northern parts of Vancouver island. For the Gulf of Mexico, we used the same geographic area as the one defined in [@Tunnell2009] by modifying manually the eastern boundaries of the shape file for the Gulf of Mexico provided by marineregions.org (mrgid:4288). These geographic areas were split into 0.2° $\times$ 0.2° quadrats that were used to query the web API of iDigBio and OBIS using the R packages ridigbio [@ridigbio-0.3.5] and robis [@robis-0.1.8] respectively. The resulting occurrences were then clipped to only include records that fell within the EEZ boundaries.

All species names obtained from all sources were checked against the World Registry of Marine Species (WoRMS). Species that could not be matched in WoRMS, and species listed as non-marine in WoRMS were not included in subsequent analyses. For all names that were listed in WoRMS, the current accepted name was capture and used in all analyses. This ensured that the same species listed under different synonyms was only counted once. The R package worrms [@worrms-0.1.0] was used to query the WoRMS database.

To assess the proportion of species for which DNA barcodes are available, we queried the Barcode of Life Datasystem (BOLD) using the R package bold [@bold-0.4.0]. We used the accepted names of species in the other lists, as well as synonyms (as listed in WoRMS) when accepted names did not return any records.

This study is fully reproducible and the code needed to perform the analyses included here is available from: https://github.com/fmichonneau/sok-marine-biodiversity, and archived on Zenodo (doi:xxx.xxx.xxxx). The data downloaded from the databases at the time of analysis are archived on Zenodo (doi:xxx.xxx.xxx)/




## References

<div id="refs"></div>


\newpage

<!-- ## Supplementary material -->
