---
#author: Fran\c{c}ois Michonneau
title: "Data-driven approaches to identifying knowledge gaps in marine invertebrate biodiversity"
date: '`r format(Sys.time())`'
output:
  bookdown::pdf_document2:
    #includes:
    #  before_body: ../template/doc_prefix.tex
    #  in_header: ../template/preamble.tex
    keep_tex: yes
    latex_engine: pdflatex
    number_sections: no
    toc: no
  bookdown::word_document2: null
fontsize: 12pt
link-citations: yes
csl: nature.csl
bibliography: /home/francois/Library/dark-plankton.bib
---

```{r setup, echo=FALSE, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```

<!-- use Pimm+2014 intro as model on using short informative sentences to build -->
<!-- 1st paragraph. -->


<!-- Species identification from DNA sequences is poised to revolutionize our understanding of biodiversity, to facilitate environmental monitoring, and to disentangle trophic networks. However, this approach relies on comprehensive databases of accurately identified sequences.  Since its introduction in the early 2000's, DNA barcoding, the sequencing of a common locus across taxa bearing species-specific information, has revitalized taxonomy by facilitating species delineation. More recently, advances in high-throughput sequencing technologies have enabled the parallel sequencing of thousands of species that occur in environmental samples. The approach initially developed to study microbial communities such as the ones from guts or soil, has been modified and extended to be applied to metazoan communities. This approach opens new research venues as it becomes possible to estimate the diversity from species-rich samples that were virtually inaccessible before. This is particularly true in marine communities that include multiple phyla and organisms varying in size by order of magnitude, making identification challenging. However, the utility of the data generated by these projects is enhanced by the possibility of matching the DNA sequences obtained to previously sequenced and correctly identified species. Here, we combined information available in public biodiversity databases to assess the proportion of identified species that have been sequenced in the continental United States. We show that the effort is consistent across the better-studied taxa, with ` average_prop_species_seq`% of the fauna sequenced. Using a DNA barcode dataset of ` n_larvae` planktonic larvae, representing ` n_esus` species, collected in coastal waters of Florida, we find a similar proportion of species that can be assigned to a previously sequenced species ( ` format_output(100 * prop_plankton_with_match)`%).  Our results show that most marine species remain to be sequenced. Achieving this goal will required a concerted effort, and resources. -->


<!--
the content of insect traps, marine sediments, plankton
samples, or communities settling on artificial substratum. When applied to
metazoan communities, these sequencing approaches typically target the barcoding
gene and therefore has been dubbed "meta-barcoding". These applications are also
making the public databases for these DNA barcodes all the more valuable. To
deliver on the meta-barcoding promises, it is crucial to have comprehensive
taxonomic coverage represented in the DNA sequences public databases. Here, we
investigate the proportion of species for each marine phylum that have been
sequenced at various spatial scale.
-->

<!-- studies that have summary statistics on what has been sequenced for various marine phyla:
- for Antarctica: Grant+Linse2009-Polar Biology:


We know that some perturbations have large effects because they are visible, but what about perturbations for which we don't see the effects directly because they affect small species? :: checkout nature paper on sea grasses.

- Describing new species is crucial but not sufficient. Without knowing their ranges, and their habitats, we can't study or predict how perturbations of the environment are going to affect their distributions and their susceptibility to perturbations. The process of figuring out the ranges of species is more time consuming than possibly finding new species; and it requires shared knowledge in the community about these new species. Using barcoding and other moleuclar techniques to improve the documentation of species distributions is going to be critical.

There can be public involvement in species description, but the exploration and collecting may remain an affair of specialists as it requires manipulating technical gear of have expertise. There are still many species that are easy to access to discover, but the bigger gap is in the species that are difficult to access, and small.

We also need to better coordination in describing new species; so that efforts are not duplicated.

Possibly also need to set guidelines on how to treat cases where the molecular data alone is too ambiguous.



-->


<!--  study focused on USA but conclusions more general given that it's arguably one of the better studied area + data availability -->
<!-- * barcoding has shown that typically marine invertebrates are complex of cryptic species with small ranges; therefore potentially more vulnerable to extinction risk. -->
<!-- * need collaboration among researchers to share data standards, protocols -->
<!-- * use existing knowledge to fill missing gaps -->
<!-- * running against time -->
<!-- * we need to document species diversity better for conservation (knowing what we protect and to measure the effectiveness of the protection) -->
<!-- * how do we conserve animal species that we can't see, we don't find pretty, don't have eyes -->
<!-- * recent advances in technologies make possibility of developing a global monitoring program possible: reduced costs of sequencing, detection of rare species, etc.... -->
<!-- * Using this kind of approaches at a large scale might also be a good way to have a better sense of how much of the biodiversity we are missing. -->
<!-- * Most species are rare, especially in very diverse places,  and the degradation of habitats makes it even more difficult to find these rare species. -->

## Abstract

With increasing impact of human activities on the biosphere, we need efficient and accurate approaches to document changes in species ranges and community composition. While terrestrial ecosystems have received considerable attention, the baseline of healthy and resilient marine ecosystems are not well understood, and modeling the impact of human activities is limited to a few taxonomic groups. A major hinderance is that we do not have enough information about which species live where, and an unknown but potentially significant portion of marine biodiversity remains to be discovered and described. Additionally, coordinated marine ecosystem monitoring initiatives are lacking [@Duffy2013,@Borja2016]. There is therefore a pressing need to improve our knowledge of marine systems before species go extinct, and their functionning are irreversibly altered.

Until very recently, our knowledge of biodiversity was scattered among heterogeneous sources (e.g. natural history collections, publications, expert knowledge), and difficult to access. These factors have contributed to the lack of synthesis and have made the identification of knowledge gaps difficult. However, with the development of public databases for natural history records (e.g. iDigBio [@Page2015]), species occurrences (e.g., GBIF [@Edwards2004], OBIS [@Zhang2002]), genetic data (e.g., GenBank [@Benson2013], BOLD [@Ratnasingham2007a]), and registers of species names (e.g., WoRMS [@Costello2013b]), we can now evaluate what we know. Here, we aggregate publicly available data for marine invertebrates that occur in the United States, in the Gulf of Mexico, and in the Pacific North-West, to highlight gaps in our knowledge. We show that public databases are relatively incomplete, and that most species are known from a few speciemens and a few locations. We emphasize that our understanding of marine biodiversity, and the impact of human activities on communities require not only to get a more accurate list of the species that inhabits the oceans, but we also need more comprehensive data on species distributions and abundances. This can only be achieved with additional sampling and a commitment to training more taxonomists and experts that can identify the material collected.

## Results

### Comparison with Regional Lists by Geography

Regional faunistic lists assembled and curated by taxonomic experts typically provide a comprehensive and detailed account of species that are known to occur in a given geographic area. We assessed how faunistic lists assembled over decades by specialists compare with records in public databases for the same geographic area. We parsed faunistic lists to extract marine invertebrates from the Gulf of Mexico [@Tunnell2009] and from the Pacific Northwest [@Kozloff1987]. Our goals were (1) to compare  the species lists for these geographic areas assembled by experts with the records found in databases; (2) to assess the proportion of species included in the expert lists that have records in public databases.

To compare the faunistic lists from the Gulf of Mexico and the Pacific Northwest with records in public databases, we queried the iDigBio and OBIS databases for all records they included in the same geographic boundaries as the lists. After filtering these lists to only include the same higher taxonomic level taxa as the species lists, and keeping only records for which the species could be match to a species name in WoRMS, we had for the Gulf of Mexico `r nrow(idigbio_gom_records)` and `r nrow(obis_gom_records)` records for iDigBio and OBIS respectively representing `r dplyr::n_distinct(idigbio_gom_records$worms_valid_name)` and `r dplyr::n_distinct(obis_gom_records$worms_valid_name)` species. For the Pacific Northwest, we retrieved `r nrow(idigbio_kozloff_records)` and `r nrow(obis_kozloff_records)` records for iDigBio and OBIS respectively representing `r dplyr::n_distinct(idigbio_kozloff_records$worms_valid_name)` and `r dplyr::n_distinct(obis_kozloff_records$worms_valid_name)` species. For the Gulf of Mexico, the majority of the species included in the expert lists are recorded in both iDigBio and OBIS (`r format_output(species_in_common$n_spp_gom_idigbio_obis/species_in_common$n_spp_gom*100)` %), or OBIS (`r format_output(species_in_common$n_spp_gom_obis/species_in_common$n_spp_gom*100)` %) (Fig. \@ref(fig:database-overlap)). However, for the Pacific Northwest, only `r format_output(species_in_common$n_spp_kozloff_idigbio_obis/species_in_common$n_spp_kozloff*100)`% and `r format_output(species_in_common$n_spp_kozloff_obis/species_in_common$n_spp_kozloff*100)`% of the species from the regional list had records in OBIS and iDigBio or OBIS respectively (Fig. 1).


```{r compare_database_overlap, fig.cap='Overlap between the regional species list (list), records in iDigBio (idigbio), records in OBIS (obis), and records in BOLD (bold) for the Gulf of Mexico (left) and the Pacific Northwest (right). (\\#fig:database-overlap)', echo=FALSE}
knitr::include_graphics("figures/database_coverage.pdf")
```


Species that are included in the list but do not have records in databases could be (1) rare: they have never been collected or have been collected only a few times, and the corresponding records have not been digitized yet; (2) difficult to preserve: large, very small, or fragile species are preserved less frequently; (3) are in the database under a different name: the species has been synonimized or re-identified since the publication of the list under a different name that may be included in the database; (4) at the limit of its range: it was included in the list because it is known to occur at the limit of its geographic zone, but has never been collected within the boundaries.

Species that are in the database but not in the list could be: (1) records not considered by the authors of the checklist: either because they were not accessible or known to the authors; (2) records erronously excluded the records (2) records that have been discovered since the list has been assembled; (3) records with incorrect identification; (4) records with incorrect geographic coordinates: their associated geographic information in the database places them in the Gulf of Mexico or the Pacific Northwest, but they were actually collected elsewhere.


### Comparison with Regional Lists by Species Names

To assess whether species included in regional lists but absent from the same geographic area nevertheless occur elsewhere, we queried the iDigBio and OBIS databases for all the marine invertebrate species included in these lists.

We extracted `r sum(gom_worms$is_marine, na.rm = TRUE)` marine invertebrates species from the faunistic list of the Gulf of Mexico, and `r sum(kozloff_worms$is_marine, na.rm=TRUE)` species from the Salish Sea. We then looked for the number of species listed that are known to occur in the same geographic area, the United States, or anywhere else in the world. Overall, OBIS had georeferenced records for most of the species that are included in the species lists. However, even for the better studied phyla about 75% of the species have records from the same geographic area, while it is less than 50% for the Pacific Northwest.

```{r database_richness, fig.cap='Comparison of the fraction of species listed in regional lists that are present in biodiversity databases for the Gulf of Mexico (A) and the Pacific Northwest (B).'}
knitr::include_graphics("figures/richness_per_db.pdf")
```

These results suggest that even when considering species lists that have been curated and that represent possibly the most accurate knowledge for this geographic area, today we could only reconstruct these lists partially based on digitized information. For instance, based on OBIS data the list would be at `r format_output(summary_richness_per_db_gom$prop_obis_in_gom[summary_richness_per_db_gom$taxon_name == "Porifera"]*100)`% complete for sponges to `r format_output(summary_richness_per_db_gom$prop_obis_in_gom[summary_richness_per_db_gom$taxon_name == "Mollusca"]*100)`% for molluscs. However, these levels drop significantly if we consider species for which museum specimens are available from  `r format_output(summary_richness_per_db_gom$prop_idigbio_in_gom[summary_richness_per_db_gom$taxon_name == "Mollusca"]*100)`% for molluscs to `r format_output(summary_richness_per_db_gom$prop_idigbio_in_gom[summary_richness_per_db_gom$taxon_name == "Annelida"]*100)`% for annelids.
It highlights that additional digitization is necessary but even more importantly additional collecting is needed to validate the occurences of the species included in these lists.

Species are unevenly represented in databases, with 25% to 50% of species listed to occur in these locations, that are not represented in digital records of museum collections. Typically a higher proportion of species from the Pacific Coast than the Gulf of Mexico are not represented. Comparatively, all but 2 of the `r nrow(mammals_species_asm)` mammal species and `r sum(amphibian_summary$count == 0)` of the `r nrow(amphibian_summary)` amphibian species known to occur in the US have digitized museum specimens.


### What do we know about marine invertebrates in the US?

We combined `r nrow(idigbio_records)` museum records for marine invertebrates collected in the EEZ waters of the continental US representing `r length(unique(tolower(idigbio_records[["data.dwc:phylum"]])))` phyla and `r length(unique(idigbio_records$worms_valid_name))` species. Most samples were collected on the continental shelf (Fig. 1). For the East coast, species richness is the highest in South Florida but it is relatively uniform along the coastline. For the West coast, the Salish Sea, the Bay Area, and Southern California have the highest richness (Fig. 2).

Based on these data, there is no evidence of a diversity gradient.

These patterns highlight our lack of knowledge about the marine fauna: `r prop_singleton_species`% of the species are known from a single record. Furthermore, the strong correlation between the number of samples collected and the species richness suggests that any new sampling effort will lead to new records (Fig. 3). This is also reflected in the cumulative numbers of species recorded for the first time and the number of samples through time (Fig. 4). The slowing down of collections in the 1970 has been accompanied with a slowing down in the discovery of new records: `r n_spp_comparison$n_spp_2017_2000` species have been documented for the first time since 2000, while `r n_spp_comparison$n_spp_1970_1960` have been documented between 1960 and 1970.

There are important discrepancies in our level of knowledge across phyla. While larger, and accessible groups are well represented in museum collections, and have a reasonable proportion of their known species represented in DNA barcode databases, the small organisms of the meiofauna are virtually absent: 12 of the 25 phyla represented in the dataset have less than 500 specimens in museums.

However, taxonomic expertise is dwindling, and there are now only a handful of US natural history museums with active marine invertebrates collections. Museum collections have been recording our knowledge about species and their geographic ranges, but until recently, this information was fragmented across institutions and difficult to synthesize. With collection digitization and other large-scale projects have aimed at gathering biodiversity information, public databases can now be used to enable synthesis and highlight knowledge gaps. Here we use these data to synthetic the state of knowledge for marine invertebrates, and demonstrate that the stage is set to combine these data sources to inform where future efforts for documenting biodiversity.


### Current genetic data and the future of biodiveristy discovery and monitoring

The proportion of species that have DNA barcodes available is surprisingly constant across the better studied phyla. About 20% of species known to occur in the Gulf of Mexico and in the Salish Sea have DNA barcode records.  Because geographic information is missing for most records (`r format_output(100*stat_bold$p_no_coords)`% of the `r format(stat_bold$n_total_records)` records analyzed here have no geographic coordinates information associated with them), and many marine invertebrate species are known to be cryptic complexes that are not as widely distributed as previously thought. Therefore specimens identified under the same species but sampled in distant locations, could represent cryptic species. It is thus crucial to ensure that accurate geographic information is provided when DNA barcodes are published, and that additional samples are collected, to capture more accurately their geographic range and the extent of their genetic diversity. This high level of cryptic diversity is reflected in the fact that `r stat_bold$n_spp_muti_bin` species out of `r stat_bold$n_spp` include more than 1 BIN. This number is confounded by some taxonomic identification issues as `r stat_bold$n_shared_bins` BINs are shared across more than one species.








## What to do next?

* Improve data quality:
  - easy: fix mispelling in identifications
  - improve identification of unidentified lots (requires taxonomists/specialists to be able to identify the specimens)
  - add and validate geographic information associated with specimens
  - develop infrastructure and tools to continuously check and validate data
    quality in databases.

Based on these results, we recommend that additional field expeditions are organized to sample more thoroughly the marine environment. The data from museum collections make clear that any additional samples will contain new species and new records. However, taxonomists are needed to identify and describe these new species. The samples collected during these expeditions need to be photographed to document live appearance that would otherwise be lost during preservation. We also need to capture metadata associated with the specimens to be able to understand better the physical characteristics of the habitats for these species. The specimens need to be sub-sampled and have tissues suitable for genomic analysis preserved, so we can sequence their DNA barcode, or other genomic resources.

<!-- Paragraph on how sampling effort has changed through time. -->

<!-- - get numbers of species described by year, look if there is a correlation between sample numbers and number of new species (possible lag with time for description) -->
<!-- - see if proportion of species identified at genus (or higher) only has increased recently -->

; and we estimate how many of the species have genetic data publicly available that would be useful in the context of identifying samples from metabarcoding studies

Using metabarcoding is a promising approach to screen an habitat for its potential for species that are not found in public database, to maximize the usefulness in the future of the data generated by an expedition. Metabarcoding approaches need to complement and take advantage of the DNA barcodes already available. Additionally, in the future, data generated by metabarcoding efforts can be re-interpreted as more DNA barcodes become available. Building the library of DNA barcodes in parallel of metabarcoding efforts also ensures that matches in the database will correspond to actual specimens deposited in natural history collections; ruling out possible artifacts of the method. The knowledge will also build on existing natural history data rather than having to restart from scratch. Beyond their utility in documenting diversity, metabarcoding is also promising for understanding the functioning of ecosystems.

## Conclusions

Given the urgency of the biodiversity crisis, we need to develop data-driven initiatives to efficiently document biodiversity, and build on our existing knowledge to collect information to improve our understanding of marine systems. Additionally,

Biodiversity discovery and documentation has been transformed in the 21st century by molecular data to facilitate species identification, delineate species. And more recently, the development of high throughput sequencing technologies is used to quantify diversity and uncover large scale patterns of biodiversity organization. The wide adoption of DNA barcoding (where homologous DNA sequences are sequenced across the main domains of the Eukaryotes) has unraveled the pervasiveness of cryptic species, invigorating taxonomy and the description of new species, and enabled collaborative research programs such as the Census of Marine Life. Unraveling these cryptic species has also contributed to the recognition that species have smaller ranges than their taxonomic identity had implied making them potentially more susceptible to extinction or overexploitation. Despite early criticisms and now well-understood limitations, DNA barcoding has been suggested as a possible solution to the widening gap between the increased need to document biodiversity while taxonomic expertise is dwindling. Thus, large databases of DNA barcodes have been collaboratively assembled and made pubicly available. By matching DNA barcodes of unidentified species to a reference public database of accurately identified and vouchered specimens, DNA barcoding can greatly speed up species identification: sequences with no match make it easier to flag specimens that warrant further taxonomic attention.

<!-- clarified our understanding of species limits in many groups, but has also enabled the identification of species during their larval stages (when the lack of morphological features make species identification difficult). DNA barcoding has also been seen as -->

Recent developments in high-throughput sequencing technologies are giving new applications to these DNA barcode databases [@Leray2016]. Metabarcoding has been developed to sequence in parallel the barcoding gene across the species included in environmental samples such as plankton tows, fish guts, or artificial structure scrappings [@Leray2015]. With decreasing sequencing costs, and the prospect of automating sample collection, these technologies will soon be used routinely for ecosystem and biodiversity monitoring [@Bohan2017]. DNA barcoding provides the possibility to link these newly generated data with historical natural history knowledge about species and their distributions. However, the insights we will gain from these approaches are only going to be as good as the level of completeness and accuracy of the information included in reference databases. Here, we mostly focus on museum specimens because we can validate their identification, and a fraction of these samples (if they have not been preserved in formalin) can be used to obtain DNA from them to complete existing databases of DNA barcodes.


<!-- Barcoding/metabarcoding certainly not perfect methods, but much better than current understanding. -->

<!-- here there are 3 things: data availability (having good APIs to access the
data); data comprehensiveness (how many species are captured in the database);
data accuracy (can we trust the data that is found in the databases) -->




![Number of species of marine invertebrates](figures/map_diversity.pdf)


![Number of museum records for marine invertebrates](figures/map_sampling_effort.pdf)

![Number of species of marine invertebrates](figures/map_diversity.pdf)

![Relation between the number of specimen and the number of species collected](figures/plot_sampling_effort.pdf)

![Cumulative number of samples through time](figures/cum_samples_through_time.pdf)

![Cumulative number of species recorded for the first time through time](figures/cum_spp_through_time.pdf)

![Proportion of species occurring in natural history collections that have been DNA barcoded](figures/proportion_barcoded.pdf)

![Relation between the number of specimen and the number of species collected](figures/plot_sampling_effort.pdf)



<!--

It has never been easier and cost effective to survey comprehensively the extant of marine biodiversity.


While plankton is recognized for its biological importance (dispersal, food source for many species), and can be used as an ecological indicator for pollution or global change, identification of planktonic larvae at the species level from morphological information is generally not possible because of the paucity of informative characters to base the identifications on, and the lack of knowledge of the appearance of the larval stages for most species. DNA barcoding can bridge this knowledge gap by using genetic information to identify larvae at the species level. These identifications are only possible if the species have already been sequenced, and the sequences deposited in public repositories. The accuracy of the identification relies on the expertise of the taxonomists who identified the specimens that have been sequenced.

We collected over the course of a year planktonic larvae of marine invertebrates in a coastal marsh of the Atlantic coast of Florida. One of our goals was to identify these larvae at the species level using DNA barcoding. We photographed and sequenced xxx larvae, representing an estimated xxx species. These data are available from BOLD and can be browsed interactively at http://plankton-atlas.net. Using records from public databases (BOLD, GenBank), we could confidently assign only xxx% to species that had already been sequenced. On average, these taxa were sequenced xxx km away from our study site.

Florida is one of the most populated states, with a long tradition of studying its marine life.  xxx universities have marine research programs, and xxx marine laboratories. How does this proportion of xxx% identified species compare with other places in Florida, the East coast of the US, the continental US, and the world. In other words, how much of the species of marine invertebrates we know about have publicly available DNA barcodes?

To estimate this number, we assembled species lists of marine invertebrates for the continental US by querying iDigBio for each marine phylum. We only kept records for specimens identified with a valid binomial name (specimens identified at the genus level or higher were excluded from the analysis). We used WoRMS to ensure that the species names were valid, and to disambiguate possible synonyms. For each species, we retrieved avaiable DNA barcoding sequence data from BOLD, which also includes DNA barcodes deposited in GenBank. Of the xxx marine invertebrate species found in iDigBio, xxx% had DNA barcodes available in public databases. These proportions varied by phylum, but for the conspicuous and better studied groups (Porifera, Cnidaria, Mollusca, Echinodermata), they were surprisingly consistent, and is a close estimate to the proportion of species that have been sequenced for the United States. Furthermore, we also looked for the proportion of species known to occur in the Indian River Lagoon for which DNA barcodes are available, and again the proportion of sequenced species (xxx%) is comparable.

Except for the few phyla that are known to include only a small number of species (Cycliophora, Phoronida, Chaetognatha), microscopic groups, typically component of the meiofauna, are disproportionally under-represented in museum collections, and none or few species are available in public databases.

Our results highlight that most of our knowledge of marine biodiversity come from large expeditions carried in the late 19th century and after World War II. Since then, sampling effort has dramatically decreased, and taxonomic expertise is waning. Yet, our ability to assign DNA sequences to species names will greatly improve the value of the data generated by meta-barcoding studies.

and reflect that a large fraction of the diversity of marine invertebrates remain to be sequenced.


Marine invertebrates are the biodiversity. All but one metazoan phyla occur in the sea. Of the xx described species, yy are marine invertebrates. They represent a multiple array of shapes, forms, and adaptations to all marine environments. Yet, they are proportionally understudied. Here, we analyze publicly available data from biodiversity databases to demonstrate that the abandonment of traditional taxonomic practices, the lack of support for natural history collections, is hampering our ability to use marine invertebrates as indicators of ecosystem health, the effect of global change, the identification of model organisms, and ... Technological advances are however making easier to identify knowledge gaps, the potential to identify where new species occur, and document diversity.


# The role of DNA barcoding

The use of molecular data for species identification and delineation is arguably one of the most important advances in taxonomy. Especially with DNA barcoding that proposes to use a common genetic marker across large taxonomic groups (the mt marker COI for animals, ITS for fungi, and xx for plants). Because these markers have typically low intra-specific variation, and differ noticeably between even closely related species, they can be used to identify species and unravel species complexes.

Species identification relies on reference databases, where sequences are associated with accurately identified individuals, held in natural history collections. When taxonomic sampling is comprehensive, and enough individuals of each species are sampled to be representative of the standing genetic variation in the population, DNA barcoding can be a very accurate tool with xx-yy% error rate.

The utility of these reference databases is getting revived with the development of meta-barcoding techniques, where thousands of species can be sequenced in parallel on high-throughput sequencing platforms. If methods exist to estimate the number of species these samples contain based on the level of genetic divergence observed among the sequences, being able to match these sequences to identified species based on their match to a sequence in a public database would increase the potential applications for these approaches.

Marine invertebrates are however disproportionally under-represented in DNA sequence databses. And they have a higher rate of misidentification. There are however some pitfalls to this number. It might actually be even higher given that they have been identified by a single taxonomist given the lack of redundancy in expertise.

To illustrate the consequences of having so few barcodes for invertebrates, if we take the faunistic list of IRL, only about 20% of the species known to be occuring in the area actually have DNA sequences available for them. On average, each species has been sequenced xx times, and from locations on average yy km away from IRL. Given the high propencity for cryptic species among marine invertebrates, it is possible that ...

We also sequenced xx larvae occurring in the plankton at the Whitney Marine Lab, representing yy species (estimated from genetic distances). Of these yy species, only zz% had good matches in BOLD/GenBank. In both cases, taxa1, taxa2, and taxa3 are the most underrepresented in BOLD.


# The waining of taxonomy

Total number of collections in the US, how many have marine invertebrate collections, how many have active curators.




We don't know how many species are in the sea. Estimates vary widely. Difficult to sample because diversity of habitats, each requiring special sampling techniques. Meanwhile, taxonomic expertise is decreasing despite efforts to keep it alive (PEET).

To get around issue of sampling different habitat, sample coastal meroplankton. Most species go through planktonic phase, therefore common habitat to sample them all. The issue is that they look very different from adults, and only DNA can be used to identify them.

Barcoding is not perfect, but can be very powerful to identify the larval forms (e.g. one of the first paper was sampling caterpillars). Potentially differences in larva that are not seen in adults and could be additional characters to tell species appart.

Advantage of sampling plankton is that a single habitat is sampled. Not enough known about heterogeneity in spatial distribution.

We sampled plankton bi-weekly for a year, and sequenced the larvae found. Sequenced the barcoding gene and tried to match how many species we could identify based on records in public databases. Overall, low percentage of match. And matches also led to several species that have been collected but that have not been identified to an appropriate level.

To compare the results we obtained here, and see how they could compare to other habitats, we downloaded the species list from Indian River lagoon inventory. It is the most comprehensive checklist in Florida, and has been developed over many years. We asked how many of the species occurring in the Indian River Lagoon have already been sequenced. We did the matching based on the species names. If we find higher proportion of sequenced species, it means that plankton gives access to species that are difficult to sample as adults. If lower proportion, it means that we don't have many of the adults sequenced at Whitney.

Using plankton to sample biodiversity could lead to a speedup of documentation. It would allow the identification of taxa/habitats that need special attention. If representative of adult communities, could also be a useful tool to monitor status of ecosystem through time.

Better database, can also be used to estimate the proportion of really dark planktonic taxa.

Understand better population connectivity, currents



Importance of plankton in connecting populations. Important as food source.

Swimming in the dark in the age of big data

Biodiversity is in the sea. All phyla but one. New lineages of eukaryotes, viruses...

We are at a time where it becomes cheaper and easier to delineate species. To sort through specimens to identify species. So taxonomists can focus on studying/describing new species.

Granularity of data not sufficient to be able to detect changes in the environment, shifts in species ranges, changes in trophic structure. Understand effect of small animals.

Rely on high school students, citizen science initiatives to help with labor intensive part.

Oceans are fascinating.

Bioprospection

focus on a few taxa but lots of groups we know nothing about.

Strong focus on vertebrates. We are building theories and making conservation approaches on a small fraction of the tree of life.

Trait data

Species Distribution



Identified specimens from museum collections could be used to complete these datasets. Traditional approaches relying on Sanger sequencing might be limiting as many species have not been collected recently enough to guarantee good quality DNA (` prop_species_not_collected_since_1990`% have not been collected since 1990). However, high-throughput sequencing could offer alternatives to recover sequences from older specimens.

Besides data-mining museum records to identify knowledge gaps, new sampling approaches can be used in the field to rapidly assess how much of the fauna is known. Meta-barcoding and environmental DNA approaches appear promising to sample _en masse_ species that are found in a given location. Because most marine invertebrates have a planktonic larval stage, they may co-occur in the water column, allowing to sample species across habitats. To illustrate this approach, we individually DNA barcoded ` n_larvae` planktonic larvae for DNA barcoding, collected over the course of year, representing ` n_esus` species, collected in coastal waters of Florida. We found that (` format_output(100 * prop_plankton_with_match)`%, Fig. 4) can be assigned to a previously sequenced species. Similar proportions have been obtained from individually barcoded species that occur on ARMS. However, this proportion dropped to less than 5% for smaller organisms sequenced with meta-barcoding.

The species identifications in BOLD are far from perfect. Of the ` sum(prop_plankton_with_species_match_by_phylum$n_esu_match)` species that have a match in public database, ` format_output(prop_plankton_with_multi_species_match * 100)`% have been identified as more than one taxon in BOLD. A few of these records might represent species complexes, but most illustrate the difficulty to identify precisely marine invertebrates.

--->
