---
#author: Fran\c{c}ois Michonneau
title: "The state of knowledge for marine invertebrate biodiversity in the continental US"
date: ' ' #'`r format(Sys.time())`'
output:
  bookdown::pdf_document2:
    #includes:
    #  before_body: ../template/doc_prefix.tex
    #  in_header: ../template/preamble.tex
    keep_tex: yes
    latex_engine: pdflatex
    number_sections: no
    toc: no
fontsize: 12pt
link-citations: yes
csl: nature.csl
bibliography: /home/francois/Library/dark-plankton.bib
---

```{r setup, echo=FALSE, include=FALSE}
knitr::opts_chunk$set(echo = FALSE)
```

<!-- use Pimm+2014 intro as model on using short informative sentences to build -->
<!-- 1st paragraph. -->



<!--



There can be public involvement in species description, but the exploration and collecting may remain an affair of specialists as it requires manipulating technical gear of have expertise. There are still many species that are easy to access to discover, but the bigger gap is in the species that are difficult to access, and small.

We also need to better coordination in describing new species; so that efforts are not duplicated.

Possibly also need to set guidelines on how to treat cases where the molecular data alone is too ambiguous.



-->


<!-- * how do we conserve animal species that we can't see, we don't find pretty, don't have eyes -->
<!-- * Most species are rare, especially in very diverse places,  and the degradation of habitats makes it even more difficult to find these rare species. -->

## Abstract

Museum collections have been recording our knowledge about species and their geographic ranges, but until recently, this information was fragmented across institutions and difficult to synthesize. The digitization of natural history collections and other biodiversity information systems can now be used to enable synthesis and highlight knowledge gaps. Here we use these data to synthesize the state of our knowledge for marine invertebrates in the continental US. We show that current knowledge and sampling efforts are inadequate given the diversity of the fauna and the geographic extent of US. While recent high-throughput sequencing technologies promise to greatly speed up the documentation of marine species and their ranges, its value would be greatly improved by anchoring the results into existing DNA sequence data and associated vouchers. However, the best documented phyla have at most 25% of their species sequenced for the most widely used DNA barcoding gene, cytochrome oxidase I (COI). More sampling, coordinated at the scale of the US, is needed to improve our knowledge of species and their geographic ranges. This data is needed to detect the impact of climate changes and other anthropogenic strains to global environments. Advances in computing, information systems, and sequencing technologies, have set the stage to inform where future efforts for documenting biodiversity should be applied.


## Introduction

With increasing impact of human activities on the biosphere, we need efficient and accurate approaches to document changes in species ranges and community composition. While terrestrial ecosystems have received considerable attention, the baseline of healthy and resilient marine ecosystems are not well understood [@Worm2006], and modeling the impact of human activities is limited to a few taxonomic groups [@Tittensor2010]. For marine organisms, because of the diversity of their complex life history strategies and their size range, we have limited information about which species live where, or what portion of their diversity remains to be discovered and described [@Mora2011]. Additionally, coordinated marine ecosystem monitoring initiatives are lacking [@Duffy2013,@Borja2016], making it difficult to monitor changes in species distributions and abundances. While most documented global extinctions of marine species are vertebrates, many others may have been lost unnoticed because they were not known [@Sala2006]. There is therefore a pressing need to improve our knowledge of marine systems before their functioning is irreversibly altered, and species go extinct.

Research attempting to understand how much we know about marine biodiversity has focused on estimating the total number of species that inhabits the oceans, and which fraction of this diversity has been named [@Appeltans2012,@Costello2013]. Undoubtely, the development of molecular methods to investigate species limits has reinvigorated taxonomy, and contributed to elucidating cryptic species complexes [@Knowlton2000]. Thus, it has been argued that molecular data can speed up the enterprise of discovering and naming species [@Blaxter2016]. As species remain the most convenient unit to measure biological diversity, understanding how much has been done, how much remains to be done, and how we can speed up the description of species diversity is crucial. However, to understand the role of these species in their environment, we need to document the ecology of these species [@Cernansky2017]. For instance, to be able to predict how their geographic ranges will be affected by global change, we need detailed information about their environmental requirements. Occurrence data for marine species could be used for environmental niche modelling of their geographic and bathymetric distributions but the proportion of species for which we have enough data to do so is unknown.

Until very recently, our knowledge of biodiversity was scattered among heterogeneous sources (e.g. natural history collections, publications, expert knowledge), difficult to access, and therefore difficult to synthesize. The development of public databases for natural history records (e.g. iDigBio [@Page2015]), species occurrences (e.g., OBIS [@Zhang2002]), genetic data (e.g., GenBank [@Benson2013], BOLD [@Ratnasingham2007a]), and registers of species names (e.g., WoRMS [@Costello2013b]), now allow us to evaluate what and how much we know. Here, we aggregate publicly available data for marine invertebrates that occur within the limits of the exclusive economic zone (EEZ) of the continental United States (US), as well as the Gulf of Mexico and the Pacific Northwest, to highlight gaps in our knowledge that provide a roadmap for important new directions of global research. We show that public databases are relatively incomplete, and that most species are known from only a few specimens and a few locations. We emphasize that our understanding of marine biodiversity, and the impact of human activities on communities require not only to get a more accurate list of the species that inhabits the oceans, but we also need more comprehensive data on species distributions and abundances. This can only be achieved with additional cost-effective sampling and a commitment to training more taxonomists and experts that can identify and interpret the material collected.

Here, we focus on the continental US because the marine biodiversity is extensively documented as the result of a long history of studies by governmental, academic, and private institutions [@Fautin2010]. The continental US includes a wide diversity of marine ecosystems from the Florida Keys, to the Gulf of Maine on the East Coast; and from the arid Southern California to the oceanic coast of the Pacific Northwest on the West coast. With estuaries, bays, sandy beaches, rocky shores, and coral reefs, the US coastlines harbors many biotas. While the continental shelf extends for several hundred kilometers on the East coast, it is narrow on the West coast. This diversity of habitats and bathymetric profiles, combined with efforts at the national level to make data publicly available, make this region especially suitable to evaluate how much we know about marine biodiversity using biodiversity databases. However, apart from the syntheses that emerged from the Census of Marine Life efforts [@Costello2010,@Fautin2010], no study has yet attempted to synthesize the current state of knowledge of marine biodiversity across the continental US, and this study is the first using data from public databases.



## Results

### Comparison with Regional Lists by Geography

Regional faunistic lists assembled and curated by taxonomic experts typically provide a comprehensive and detailed account of species that are known to occur in a given geographic area. We assessed how faunistic lists assembled over decades by specialists compare with records in public databases for the same geographic area. We parsed faunistic lists to extract marine invertebrates from the Gulf of Mexico [@Tunnell2009] and from the Pacific Northwest [@Kozloff1987]. Our goals were (1) to compare  the species lists for these geographic areas assembled by experts with the records found in databases; (2) to assess the proportion of species included in the expert lists that have records in public databases.

To compare the faunistic lists from the Gulf of Mexico and the Pacific Northwest with records in public databases, we queried the iDigBio (https://portal.idigbio.org) and OBIS (http://www.iobis.org/) databases for all records they included in the same geographic boundaries as the lists. After filtering these lists to only include the same higher taxonomic level taxa as the species lists, and keeping only records for which the species could be matched to a species name in WoRMS, we had for the Gulf of Mexico `r format(nrow(idigbio_gom_records), big.mark=",")` and `r format(nrow(obis_gom_records), big.mark=",")` records for iDigBio and OBIS respectively, representing `r format(dplyr::n_distinct(idigbio_gom_records$worms_valid_name), big.mark=",")` and `r format(dplyr::n_distinct(obis_gom_records$worms_valid_name), big.mark=",")` species. For the Pacific Northwest, we retrieved `r format(nrow(idigbio_pnw_records), big.mark=",")` and `r format(nrow(obis_kozloff_records), big.mark=",")` records for iDigBio and OBIS respectively representing `r format(dplyr::n_distinct(idigbio_pnw_records$worms_valid_name), big.mark=",")` and `r format(dplyr::n_distinct(obis_kozloff_records$worms_valid_name), big.mark=",")` species. For the Gulf of Mexico, the majority of the species included in the expert lists are recorded in both iDigBio and OBIS (`r format_output(species_in_common$n_spp_gom_idigbio_obis/species_in_common$n_spp_gom*100)` %), or only OBIS (`r format_output(species_in_common$n_spp_gom_obis/species_in_common$n_spp_gom*100)` %) (Fig. \@ref(fig:database-overlap)). For the Pacific Northwest, these proportions were much lower with only `r format_output(species_in_common$n_spp_kozloff_idigbio_obis/species_in_common$n_spp_kozloff*100)`% of the species from the regional list with records in OBIS and iDigBio, and `r format_output(species_in_common$n_spp_kozloff_obis/species_in_common$n_spp_kozloff*100)`%  in OBIS only (Fig. \@ref(fig:database-coverage)). Previous esimates for species richness in the Gulf of Mexico for marine invertebrates were of `r format(11150-1541-434, big.mark=",")` species, and `r format(8374-909-134, big.mark=",")` for the Pacific coast (including California) [@Fautin2010].

```{r compare_database_coverage, fig.cap='Overlap between the regional species list (list), records in iDigBio (idigbio), records in OBIS (obis), and records in BOLD (bold) for the Gulf of Mexico (left) and the Pacific Northwest (right). (\\#fig:database-coverage)', echo=FALSE}
knitr::include_graphics("figures/database_coverage.pdf")
```

We investigated the number of species for each phylum that contributed to the discrepancies between the regional species lists and the database records. We found that the better studied phyla contributed the most to the differences across the data sources. OBIS tends to be closer to the full record set, but iDigBio has more species for the Gulf of Mexico that are not included in the list (Fig. \@ref(fig:database-overlap))


Overall, there is more overlap among the data sources for the Gulf of Mexico than the Pacific Northwest.




```{r compare_database_overlap, fig.cap='Comparison of species by phylum across the public databases and the faunistic list. (\\#fig:database-overlap)', ech=FALSE}
knitr::include_graphics("figures/database_overlap.pdf")
```


Species that are included in the list but do not have records in databases could be (1) rare: they have never been collected or have been collected only a few times, and the corresponding records have not been digitized yet; (2) difficult to preserve: large, very small, or fragile species are preserved less frequently; (3) are in the database under a different name: the species has been synonimized or re-identified since the publication of the list under a different name that may be included in the database; (4) at the limit of its range: it was included in the list because it is known to occur at the limit of its geographic zone, but has never been collected within the boundaries; (5) does not have geographic coordinates associated with it: it can be in the database but without georeferenced datapoints that made it possible to discover them. By querying iDigBio we found that for the `r format(not_really_in_database$idig_gom_n_spp_not_in_db, big.mark=",")` species that do not appear within the geographic range of the Gulf of Mexico but are on the list, `r format_output(not_really_in_database$idig_gom_p_spp_not_in_db*100)`% are truly absent from iDigBio, while for the `r format(not_really_in_database$idig_gom_n_spp_no_coords, big.mark=",")` species that appear in iDigBio `r format_output(not_really_in_database$idig_gom_p_spp_no_coords*100)`%  do not have geographic coordinates associated with them, and for the ones that dok `r format_output(not_really_in_database$idig_gom_p_spp_within_usa*100)`% fall within the US waters (n = `r format(not_really_in_database$idig_gom_n_spp_within_usa, big.mark=",")`).

Species that are in the database but not in the list could be: (1) records not considered by the authors of the checklist: either because they were not accessible or known to the authors; (2) records erroneously excluded the records (2) records that have been discovered since the list has been assembled (for iDigbio records from the Gulf of Mexico, this represent `r format_output(mean(recently_collected$gom_in_idig$min_year > 2001))`%, n=`r nrow(recently_collected$gom_in_idig)` and `r format_output(mean(recently_collected$pnw_in_idig$min_year > 1987))`%, n=`r nrow(recently_collected$pnw_in_idig)`); (3) records with incorrect identification; (4) records with incorrect geographic coordinates: their associated geographic information in the database places them in the Gulf of Mexico or the Pacific Northwest, but they were actually collected elsewhere.


### Comparison with Regional Lists by Species Names

To assess whether species included in regional lists but absent from the same geographic area nevertheless occur elsewhere, we queried the iDigBio and OBIS databases for all the marine invertebrate species included in these lists.

We extracted `r format(sum(gom_worms$is_marine, na.rm = TRUE), big.mark=",")` marine invertebrates species from the faunistic list of the Gulf of Mexico, and `r format(sum(kozloff_worms$is_marine, na.rm=TRUE), big.mark=",")` species from the Pacific Northwest. We then looked for the number of species listed that are known to occur in the same geographic area, the United States, or anywhere else in the world. Overall, OBIS had georeferenced records for most of the species that are included in the species lists. However, even for the better studied phyla about 75% of the species have records from the same geographic area, while it is less than 50% for the Pacific Northwest.

```{r database_richness, fig.cap='Comparison of the fraction of species listed in regional lists that are present in biodiversity databases for the Gulf of Mexico (A) and the Pacific Northwest (B).'}
knitr::include_graphics("figures/richness_per_db.pdf")
```

These results suggest that even when considering species lists that have been curated and that represent possibly the most accurate knowledge for any particular geographic area, today we can only partially reconstruct these lists based on digitized information. For instance, based on OBIS data the list would be `r format_output(summary_richness_per_db_gom$prop_obis_in_gom[summary_richness_per_db_gom$taxon_name == "Porifera"]*100)`% complete for sponges to `r format_output(summary_richness_per_db_gom$prop_obis_in_gom[summary_richness_per_db_gom$taxon_name == "Mollusca"]*100)`% for molluscs. However, these levels drop significantly if we consider species for which museum specimens are available from  `r format_output(summary_richness_per_db_gom$prop_idigbio_in_gom[summary_richness_per_db_gom$taxon_name == "Mollusca"]*100)`% for molluscs to `r format_output(summary_richness_per_db_gom$prop_idigbio_in_gom[summary_richness_per_db_gom$taxon_name == "Annelida"]*100)`% for annelids. Similar patterns can be observed for the Pacific Northwest, but overall the levels of data availability are lower: from `r format_output(summary_richness_per_db_koz$prop_obis_in_pnw[summary_richness_per_db_koz$taxon_name == "Porifera"]*100)`% for sponges to `r format_output(summary_richness_per_db_koz$prop_obis_in_pnw[summary_richness_per_db_koz$taxon_name == "Annelida"]*100)`% for annelids for the proportion of species that have records in OBIS, and from  `r format_output(summary_richness_per_db_koz$prop_idigbio_in_pnw[summary_richness_per_db_koz$taxon_name == "Porifera"]*100)`% for sponges to `r format_output(summary_richness_per_db_koz$prop_idigbio_in_pnw[summary_richness_per_db_koz$taxon_name == "Mollusca"]*100)`% for molluscs for the proportion of species that have records in iDigBio.

Museum specimens are critical to validate the occurrence of species [@Rocha2014], especially for marine invertebrates for which species limits based on morphology have been known to be challenged by other lines of evidence including molecular data [@Knowlton2000]. Some of the discrepancies observed between the number of species for which occurrence data and museum specimens are available suggest that additional digitization of collections (including capturing their geographic information) are needed. For instance, one can expect that the digitization coverage for the Gulf of Mexico will greatly increase when records from the Florida Wildlife Research Institute collection (St Petersburg, FL) become integrated into the biodiversity information systems.


### What do we know about marine invertebrates in the US?

We combined `r format(nrow(idigbio_records), big.mark=",")` museum records for marine invertebrates collected in the EEZ waters of the continental US representing `r length(unique(tolower(idigbio_records[["data.dwc:phylum"]])))` phyla and `r format(length(unique(idigbio_records$worms_valid_name)), big.mark=",")` species. Most samples were collected on the continental shelf (Fig. \@ref(fig:map-diversity-sampling)A). For the East coast, species richness is the highest in South Florida but it is relatively uniform along the coastline. For the West coast, the Pacific Northwest, the San Francisco Bay Area, and Southern California have the highest richness (Fig. \@ref(fig:map-diversity-sampling)B).

```{r map-diversity-sampling, fig.cap='Spatial distribution of the number of samples (A) and species richness (B) for marine invertebrates based on specimen data available from iDigBio within the limits of the Exclusive Economic Zone for the continental USA. (\\#fig:map-diversity-sampling)'}
knitr::include_graphics("figures/map_diversity_sampling.pdf")
```

These patterns of spatial distribution of the species richness are strongly confounded by the sampling effort. When investigating the relation between the number of samples and the number of species for each cell of the map (Fig. \@ref(fig:correlation-sampling)), we can see that until reaching about 100 records per cell there is little redundancy in the sampled species (each record represent a different species).

```{r correlation-sampling, fig.cap='Relation between number of records and number of species for each cell of the map used in Fig. \\@ref(fig:map-diversity-sampling). (\\#fig:correlation-sampling)'}
knitr::include_graphics("figures/plot_sampling_effort.pdf")
```

The inadequacy of the sampling effort compared to the diversity of the fauna is also reflected by the fact that only `r prop_singleton_species`% of the species are known from a single record. The strong correlation between the number of samples collected and the species richness suggests that any new sampling effort will lead to new observations (Fig. \@ref(fig:map-diversity-sampling)) and new species. Another indication of this inadequacy is the relation between the number of collected specimen and the number of species recorded through time (Fig. \@ref(fig:idigbio-through-time)). Initial explorations in the late ninetieth century lead to an initial exponential growth in the knowledge of the fauna, that tapered off, until the 1970's when intensive collecting efforts led to an increase in the recording of new taxa (Fig. \@ref(fig:idigbio-through-time)). Since then, the slowing down of collections after the 1970s has been accompanied with a slowing down in the discovery of new records: `r n_spp_comparison$n_spp_2017_2000` species have been documented for the first time since 2000 (`r format_output(n_spp_comparison$n_spp_2017_2000/18)` species.year^-1^), while `r n_spp_comparison$n_spp_1970_1960` have been documented between 1960 and 1970 (`r format_output(n_spp_comparison$n_spp_1970_1960/11)` species.year^-1^). Therefore, coordinated large-scale biodiversity expeditions will continue to generate a wealth of data and new records. For instance, a recent study in Western Australia reported that 135 samples led to 805 species of crustaceans and polychaetes, 94.6% and 72% of which were undescribed [@Poore2014].

```{r idigbio-through-time, fig.cap='Cumulative number of samples (A) and species (B) of marine invertebrates recorded through time based on iDigBio records. (\\#fig:idigbio-through-time)'}
knitr::include_graphics("figures/idigbio_through_time.pdf")
```

Predicting the geographic range of species, and how they can be impacted by global change has undergone important advances recently. However, it is generally recognized that to obtain reliable results such efforts require to have accurate geographic information for a minimum of 10 specimens but having at least 30 is preferred. Based on iDigBio data, `r prop_10_specimens_species`% of species have less than 10 specimens and `r prop_30_specimens_species`% have less than 30 specimens. This suggests that our understanding of the geographic range of species is limited as most species are known from only a few locations, while efforts to estimate their current and future ranges based on occurrence data are rendered moot by the low number of data points available.

Furthermore, there are important discrepancies in our level of knowledge across phyla. While larger, and accessible groups represent the bulk of the records for marine invertebrates in museum collections, the small organisms, including representatives of the meiofauna are virtually absent. Of the `r n_idigbio_phyla` phyla occurring in iDigBio, `r records_rare_phyla$n_rare_phyla` are represented by less than 500 records, and make up `r records_rare_phyla$prop_rare_phyla`% of the total number of records.

<!-- From Mark: These previously undersampled taxa are likely to be much more important for the detection of subtle changes in environmental quality due to their size and physiological vulnerability. (?).  -->
<!-- also, the impact of changes in the composition in their communties is largely unexplored, look into Bik's paper? -->

<!-- - get numbers of species described by year, look if there is a correlation between sample numbers and number of new species (possible lag with time for description) -->
<!-- - see if proportion of species identified at genus (or higher) only has increased recently -->


### Molecular data for the future of biodiversity discovery and monitoring

Since its introduction in the early 2000's, DNA barcoding, the sequencing of a common locus across taxa bearing species-specific information, has revitalized taxonomy by facilitating species delineation [@Meyer2005-plos]. More recently, advances in high-throughput sequencing technologies have enabled the parallel sequencing of thousands of species that occur in environmental samples cost effectively [@Leray2015]. This approach opens up new research venues as it becomes possible to estimate the diversity from species-rich samples that were virtually inaccessible before. This is particularly true in marine communities that include multiple phyla and organisms varying in size by order of magnitude, making identification challenging. However, the utility of the data generated by these projects is enhanced by the possibility of matching DNA sequences obtained to previously sequenced and correctly identified species [@Leray2015]. With decreasing sequencing costs, and the prospect of automating sample collection, these technologies may soon be used routinely to monitor biodiversity at defined temporal frequency [@Bohan2017]. DNA barcoding provides the possibility to link these newly generated data with historical natural history knowledge about species and their distributions. However, the insights we will gain from these approaches are only going to be as good as the level of completeness and accuracy of the information included in reference databases.

To estimate the proportion of the marine invertebrate diversity for which DNA barcodes are available, we queried BOLD (Barcode of Life Datasystem) for all the species that are listed in the regional lists for the Gulf of Mexico, the Pacific Northwest, as well as all the records for marine invertebrates that occur in the EEZ of the US found in iDigBio. Here, we focused on records based on museum specimens as some of them could potentially be used to complete current DNA barcoding efforts.

The proportion of species that have DNA barcodes available is relatively constant across the better studied phyla at the scale of the EEZ. It varies between 12% for bryozoans to 28% for cnidarians. The proportions for species known to occur in the Gulf of Mexico are usually similar than for the US, while typically a higher fraction of species from the Pacific Northwest are available. In 2012, it was estimated that a global scale, less than 3% of the marine species had been sequenced [@Appeltans2012], and that this number was even lower in some areas (e.g. 2.6% for the Antarctic [@Grant2009]). This suggests that even as many additional species have had their DNA barcodes made available since these studies, the US fauna is generally better characterized than other parts of the world.

Using the fraction of known species that have a barcode as a yard stick of the progress made is not sufficient. Many marine invertebrate species are known to be cryptic complexes that are not as widely distributed as currently reported [@Knowlton2000,@Meyer2005-evolution]. Therefore specimens identified under the same species but sampled in distant locations, could represent cryptic species. Geographic information is missing for most records (`r format_output(100*stat_bold$p_no_coords)`% of the `r format(stat_bold$n_total_records, big.mark=",")` BOLD records analyzed here have no geographic coordinate information associated with them). It is thus crucial to ensure that accurate geographic information is provided when DNA barcodes are published, and that enough samples for each species are sequenced, to capture more accurately their geographic range and the extent of their genetic diversity. This cryptic diversity is also at least partially reflected in the fact that `r stat_bold$n_spp_multi_bin` species out of `r format(stat_bold$n_spp, big.mark=",")` include more than 1 BIN (BOLD Index Number, operational taxonomic units that closely resemble species [@Ratnasingham2013]).


```{r proportion-barcoded, fig.cap='Proportion of species known to occur in the US EEZ based on iDigBio records, in the Gulf of Mexico [@Tunnell2009], in the Pacific Northwest[@Kozloff1987], that have publicly available DNA barcodes. Numbers next to the bars indicate the number of species.'}
knitr::include_graphics("figures/proportion_barcoded.pdf")
```


## Where do we go from there?

Based on the results of this study, we recommend a multi-level approach for both improving the quality of existing data, as well as for the efficient collection of new data.

### Improve existing data

With data aggregation of natural history collections through initiatives like iDigBio, and the compilation of a nearly complete register of marine species with WoRMS, in combination with the development of algorithms for text analysis, it is increasingly easier to validate and spell check the scientific names of the identifications of digitized museum specimens. Linking species names to identifiers in taxonomic databases would facilitate data linking, specimen discoverability, and data usability. Additionally, because species names registers like WoRMS link the history of the names and their synonymies, it is possible to dissociate the concepts of species identification with the concept of species identity, allowing researchers to compare specimens identified at different time periods, a tedious process that would have required expert knowledge in the past.

Similarly, a significant fraction of museum records do not have or have inaccurate geographic information associated with them. While tools have been developed to convert names of locales into GPS coordinates on land [@rios2010geolocate], similar approaches may not work for locales at sea. However, geographic information is available in other forms, and can be digitized.

For both the spelling of scientific names, and the geographic information, computing tools can be developed to detect issues and propose solutions. There are opportunities for services using publicly available data to be developed to make curators and collection managers aware of these issues so that they can be fixed at the source. It however requires that resources are made available to museum collections to address these issues.

Increasing the computing literacy of the staff of museums in order for them to manipulate and use the data from their collection database is equally critical. As data across institutions get pooled into the large biodiversity information systems, it is crucial that data standards such as Darwin Core are embraced by the community.

Another venue to improve the quality of the data in biodiversity information systems, is better incitives from journal editors and funding agencies for scientists to publish their occurrence records in databases alongside their papers as it is done for genetic data with GenBank [@Huang2011e].

Specimens with accurate identifications and specimens that have been preserved in conditions such that it is possible to get DNA out of them should be prioritized, as they yield the most data that are relevant to biodiversity knowledge.

<!-- - develop infrastructure and tools to continuously check and validate data quality in databases. -->

### Collect better data faster and more cost effectively

This study highlights that we have too little data about too few species of marine invertebrates, we recommend that additional coordinated field expeditions are organized to sample more thoroughly the marine environment. The relation between sampling effort and diversity from museum collections makes clear that any additional samples will improve greatly our knowledge of marine biodiversity. Concerted and collaborative efforts are needed to share expertise and limited resources.

Samples collected during these expeditions need to be photographed to document live appearance that would otherwise be lost during preservation. We also need to capture metadata associated with the specimens to be able to understand better the physical characteristics of the habitats for these species. Specimens need to be sub-sampled to create material suitable for genomic analysis. Parts of data acquisition associated with specimens and their biota can be automated and captured digitally directly in the field, speeding up their ingestion in database collections.

Integrating metabarcoding as part of the data collection conducted in biodiversity surveys is a promising approach to screen an habitat for its potential to lead to new species or species that have not yet public DNA barcodes. For instance, by analyzing metabarcoding samples that contain planktonic larvae from a given habitat, we could get an indication of which taxonomic group would benefit most from additional sampling effort or give an indication of the proportion of species that has not been sampled yet. Additionally, data generated by metabarcoding efforts can be re-interpreted as more DNA barcodes associated with correctly identified specimens become available. Building the library of DNA barcodes in parallel of metabarcoding efforts ensures that database matches correspond to actual specimens deposited in natural history collections, ruling out possible artifacts of the method. With such approaches, the geographic distribution of species can build on and linked to existing records rather than being limited to matching DNA sequences across studies.

Assembling species lists is a necessary but not sufficient step in our quest to understand biodiversity. We also need to understand better the geographic distribution of biodiversity, how the abundance of species vary in space and time, and their role in the functionning of ecosystems. In order to predict the impact of climate change and other anthropogenic perturbations on species distributions, we need to collect additional occurences for species as well as the characteristics of their habitats. Among the challenges for this enterprise, we need to be able to convey taxonomic concepts for species which, for the most part, have not formally been named yet. Developing robust identifiers that can be shared across biodiversity information systems, and anchored in results from DNA barcoding or other molecular techniques, is needed to collaboratively build our knowledge of species distributions.

Metabarcoding also offers to be a promising technology to monitor changes in communities and for understanding the functioning of ecosystems. For instance, despite our limited knowledge of the diversity of communities in the meiofauna, metabarcoding has been successfully used to detect and documents changes in species assemblages following the Deepwater Horizon Oil Spill [@Bik2012]. Metabarcoding has also revealed important differences in the diet of 3 related species of fishes that co-occur at small spatial scales, providing a much more detailed understanding of trophic relations [@Leray2015b].

Considering additional linked data with species such as DNA has been a first step in a digital biodiversity world where other data on species can be gathered in databases and analyzed to better understand species distributions. This has paved the way to more linked data/databases. Still issues though with connecting identifiers [@Guralnick2014].

Even if technological advances can greatly speed up the documentation of species and their distributions, taxonomists and experts are needed  to collect, sort, process, and identify samples. Taxonomic expertise within the US is dwindling, and there are now only a handful of US natural history museums with active marine invertebrates collections, and curatorial staff.

Given the urgency of the biodiversity crisis, we need to develop data-driven initiatives to efficiently document biodiversity, and build on our existing knowledge to collect information to improve our understanding of marine systems.
Digitization efforts are catching up on an already fragmented and vastly incomplete records. Here, we only reported results from the US where there is at least two centuries of biodiversity exploration data and high-level of digitization. Many other parts of the world also need additional attention, especially in some developing countries where environmental pressure on ecosystems is even higher than in the US.


<!-- Biodiversity discovery and documentation has been transformed in the 21st century by molecular data to facilitate species identification, delineate species. And more recently, the development of high throughput sequencing technologies is used to quantify diversity and uncover large scale patterns of biodiversity organization. The wide adoption of DNA barcoding (where homologous DNA sequences are sequenced across the main domains of the Eukaryotes) has unraveled the pervasiveness of cryptic species, invigorating taxonomy and the description of new species, and enabled collaborative research programs such as the Census of Marine Life. Unraveling these cryptic species has also contributed to the recognition that species have smaller ranges than their taxonomic identity had implied making them potentially more susceptible to extinction or overexploitation. Despite early criticisms and now well-understood limitations, DNA barcoding has been suggested as a possible solution to the widening gap between the increased need to document biodiversity while taxonomic expertise is dwindling. Thus, large databases of DNA barcodes have been collaboratively assembled and made pubicly available. By matching DNA barcodes of unidentified species to a reference public database of accurately identified and vouchered specimens, DNA barcoding can greatly speed up species identification: sequences with no match make it easier to flag specimens that warrant further taxonomic attention. -->

<!-- clarified our understanding of species limits in many groups, but has also enabled the identification of species during their larval stages (when the lack of morphological features make species identification difficult). DNA barcoding has also been seen as -->

<!-- Barcoding/metabarcoding certainly not perfect methods, but much better than current understanding. -->

<!-- widening gap with other groups (vertebrates, plants) for which more comprehensive data is available -->

<!-- here there are 3 things: data availability (having good APIs to access the
data); data comprehensiveness (how many species are captured in the database);
data accuracy (can we trust the data that is found in the databases) -->


## Material and methods

### Data sources

iDigBio:
OBIS:
WoRMS:

## References


<!--

It has never been easier and cost effective to survey comprehensively the extant of marine biodiversity.


While plankton is recognized for its biological importance (dispersal, food source for many species), and can be used as an ecological indicator for pollution or global change, identification of planktonic larvae at the species level from morphological information is generally not possible because of the paucity of informative characters to base the identifications on, and the lack of knowledge of the appearance of the larval stages for most species. DNA barcoding can bridge this knowledge gap by using genetic information to identify larvae at the species level. These identifications are only possible if the species have already been sequenced, and the sequences deposited in public repositories. The accuracy of the identification relies on the expertise of the taxonomists who identified the specimens that have been sequenced.

We collected over the course of a year planktonic larvae of marine invertebrates in a coastal marsh of the Atlantic coast of Florida. One of our goals was to identify these larvae at the species level using DNA barcoding. We photographed and sequenced xxx larvae, representing an estimated xxx species. These data are available from BOLD and can be browsed interactively at http://plankton-atlas.net. Using records from public databases (BOLD, GenBank), we could confidently assign only xxx% to species that had already been sequenced. On average, these taxa were sequenced xxx km away from our study site.

Florida is one of the most populated states, with a long tradition of studying its marine life.  xxx universities have marine research programs, and xxx marine laboratories. How does this proportion of xxx% identified species compare with other places in Florida, the East coast of the US, the continental US, and the world. In other words, how much of the species of marine invertebrates we know about have publicly available DNA barcodes?

To estimate this number, we assembled species lists of marine invertebrates for the continental US by querying iDigBio for each marine phylum. We only kept records for specimens identified with a valid binomial name (specimens identified at the genus level or higher were excluded from the analysis). We used WoRMS to ensure that the species names were valid, and to disambiguate possible synonyms. For each species, we retrieved avaiable DNA barcoding sequence data from BOLD, which also includes DNA barcodes deposited in GenBank. Of the xxx marine invertebrate species found in iDigBio, xxx% had DNA barcodes available in public databases. These proportions varied by phylum, but for the conspicuous and better studied groups (Porifera, Cnidaria, Mollusca, Echinodermata), they were surprisingly consistent, and is a close estimate to the proportion of species that have been sequenced for the United States. Furthermore, we also looked for the proportion of species known to occur in the Indian River Lagoon for which DNA barcodes are available, and again the proportion of sequenced species (xxx%) is comparable.


Marine invertebrates are the biodiversity. All but one metazoan phyla occur in the sea. Of the xx described species, yy are marine invertebrates. They represent a multiple array of shapes, forms, and adaptations to all marine environments. Yet, they are proportionally understudied. Here, we analyze publicly available data from biodiversity databases to demonstrate that the abandonment of traditional taxonomic practices, the lack of support for natural history collections, is hampering our ability to use marine invertebrates as indicators of ecosystem health, the effect of global change, the identification of model organisms, and ... Technological advances are however making easier to identify knowledge gaps, the potential to identify where new species occur, and document diversity.


# The role of DNA barcoding

The use of molecular data for species identification and delineation is arguably one of the most important advances in taxonomy. Especially with DNA barcoding that proposes to use a common genetic marker across large taxonomic groups (the mt marker COI for animals, ITS for fungi, and xx for plants). Because these markers have typically low intra-specific variation, and differ noticeably between even closely related species, they can be used to identify species and unravel species complexes.

Species identification relies on reference databases, where sequences are associated with accurately identified individuals, held in natural history collections. When taxonomic sampling is comprehensive, and enough individuals of each species are sampled to be representative of the standing genetic variation in the population, DNA barcoding can be a very accurate tool with xx-yy% error rate.

The utility of these reference databases is getting revived with the development of meta-barcoding techniques, where thousands of species can be sequenced in parallel on high-throughput sequencing platforms. If methods exist to estimate the number of species these samples contain based on the level of genetic divergence observed among the sequences, being able to match these sequences to identified species based on their match to a sequence in a public database would increase the potential applications for these approaches.

To illustrate the consequences of having so few barcodes for invertebrates, if we take the faunistic list of IRL, only about 20% of the species known to be occuring in the area actually have DNA sequences available for them. On average, each species has been sequenced xx times, and from locations on average yy km away from IRL. Given the high propencity for cryptic species among marine invertebrates, it is possible that ...


# The waining of taxonomy

We don't know how many species are in the sea. Estimates vary widely. Difficult to sample because diversity of habitats, each requiring special sampling techniques. Meanwhile, taxonomic expertise is decreasing despite efforts to keep it alive (PEET).

To get around issue of sampling different habitat, sample coastal meroplankton. Most species go through planktonic phase, therefore common habitat to sample them all. The issue is that they look very different from adults, and only DNA can be used to identify them.

Barcoding is not perfect, but can be very powerful to identify the larval forms (e.g. one of the first paper was sampling caterpillars). Potentially differences in larva that are not seen in adults and could be additional characters to tell species appart.

Advantage of sampling plankton is that a single habitat is sampled. Not enough known about heterogeneity in spatial distribution.

We sampled plankton bi-weekly for a year, and sequenced the larvae found. Sequenced the barcoding gene and tried to match how many species we could identify based on records in public databases. Overall, low percentage of match. And matches also led to several species that have been collected but that have not been identified to an appropriate level.

To compare the results we obtained here, and see how they could compare to other habitats, we downloaded the species list from Indian River lagoon inventory. It is the most comprehensive checklist in Florida, and has been developed over many years. We asked how many of the species occurring in the Indian River Lagoon have already been sequenced. We did the matching based on the species names. If we find higher proportion of sequenced species, it means that plankton gives access to species that are difficult to sample as adults. If lower proportion, it means that we don't have many of the adults sequenced at Whitney.

Using plankton to sample biodiversity could lead to a speedup of documentation. It would allow the identification of taxa/habitats that need special attention. If representative of adult communities, could also be a useful tool to monitor status of ecosystem through time.

Better database, can also be used to estimate the proportion of really dark planktonic taxa.

Understand better population connectivity, currents



Importance of plankton in connecting populations. Important as food source.

Swimming in the dark in the age of big data

Biodiversity is in the sea. All phyla but one. New lineages of eukaryotes, viruses...

We are at a time where it becomes cheaper and easier to delineate species. To sort through specimens to identify species. So taxonomists can focus on studying/describing new species.

Granularity of data not sufficient to be able to detect changes in the environment, shifts in species ranges, changes in trophic structure. Understand effect of small animals.

Rely on high school students, citizen science initiatives to help with labor intensive part.

Oceans are fascinating.

Bioprospection

focus on a few taxa but lots of groups we know nothing about.

Strong focus on vertebrates. We are building theories and making conservation approaches on a small fraction of the tree of life.

Trait data

Species Distribution



Identified specimens from museum collections could be used to complete these datasets. Traditional approaches relying on Sanger sequencing might be limiting as many species have not been collected recently enough to guarantee good quality DNA (` prop_species_not_collected_since_1990`% have not been collected since 1990). However, high-throughput sequencing could offer alternatives to recover sequences from older specimens.

Besides data-mining museum records to identify knowledge gaps, new sampling approaches can be used in the field to rapidly assess how much of the fauna is known. Meta-barcoding and environmental DNA approaches appear promising to sample _en masse_ species that are found in a given location. Because most marine invertebrates have a planktonic larval stage, they may co-occur in the water column, allowing to sample species across habitats. To illustrate this approach, we individually DNA barcoded ` n_larvae` planktonic larvae for DNA barcoding, collected over the course of year, representing ` n_esus` species, collected in coastal waters of Florida. We found that (` format_output(100 * prop_plankton_with_match)`%, Fig. 4) can be assigned to a previously sequenced species. Similar proportions have been obtained from individually barcoded species that occur on ARMS. However, this proportion dropped to less than 5% for smaller organisms sequenced with meta-barcoding.

The species identifications in BOLD are far from perfect. Of the ` sum(prop_plankton_with_species_match_by_phylum$n_esu_match)` species that have a match in public database, ` format_output(prop_plankton_with_multi_species_match * 100)`% have been identified as more than one taxon in BOLD. A few of these records might represent species complexes, but most illustrate the difficulty to identify precisely marine invertebrates.

--->
